# adapr

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/adapr
* BugReports: https://github.com/gelfondjal/adapr/issues
* Date/Publication: 2017-11-30 00:19:12 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"adapr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘shinydashboard’
      All declared Imports should be used.
    Missing or unexported object: ‘devtools::clean_source’
    ```

# adegenet

<details>

* Version: 2.1.1
* Source code: https://github.com/cran/adegenet
* URL: https://github.com/thibautjombart/adegenet
* Date/Publication: 2018-02-02 18:21:36 UTC
* Number of recursive dependencies: 85

Run `revdep_details(,"adegenet")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R       2.0Mb
        data    1.3Mb
        files   1.7Mb
    ```

# AdhereR

<details>

* Version: 0.4.1
* Source code: https://github.com/cran/AdhereR
* URL: https://github.com/ddediu/AdhereR
* Date/Publication: 2019-02-11 13:43:17 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"AdhereR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        doc   3.2Mb
    ```

# analysisPipelines

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/analysisPipelines
* URL: https://github.com/Mu-Sigma/analysis-pipelines
* BugReports: https://github.com/Mu-Sigma/analysis-pipelines/issues
* Date/Publication: 2019-01-08 16:40:12 UTC
* Number of recursive dependencies: 135

Run `revdep_details(,"analysisPipelines")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘SparkR’ ‘reticulate’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘devtools’
      All declared Imports should be used.
    ```

# appreci8R

<details>

* Version: 1.0.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 127

Run `revdep_details(,"appreci8R")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      ‘BSgenome.Hsapiens.UCSC.hg19’ ‘Homo.sapiens’
      ‘SNPlocs.Hsapiens.dbSNP144.GRCh37’
      ‘XtraSNPlocs.Hsapiens.dbSNP144.GRCh37’
      ‘MafDb.1Kgenomes.phase3.hs37d5’ ‘MafDb.ExAC.r1.0.hs37d5’
      ‘MafDb.ESP6500SI.V2.SSA137.hs37d5’ ‘MafDb.gnomADex.r2.0.1.hs37d5’
      ‘PolyPhen.Hsapiens.dbSNP131’ ‘SIFT.Hsapiens.dbSNP137’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ArchaeoPhases

<details>

* Version: 1.4
* Source code: https://github.com/cran/ArchaeoPhases
* BugReports: https://github.com/marieannevibet/ArchaeoPhases/issues
* Date/Publication: 2019-01-02 09:10:03 UTC
* Number of recursive dependencies: 93

Run `revdep_details(,"ArchaeoPhases")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# archivist

<details>

* Version: 2.3.2
* Source code: https://github.com/cran/archivist
* URL: https://pbiecek.github.io/archivist/
* BugReports: https://github.com/pbiecek/archivist/issues
* Date/Publication: 2019-01-02 11:30:09 UTC
* Number of recursive dependencies: 93

Run `revdep_details(,"archivist")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘archivist.github’
    ```

# arena2r

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/arena2r
* URL: https://github.com/pedroliman/arena2r
* BugReports: https://github.com/pedroliman/arena2r/issues
* Date/Publication: 2018-10-19 15:30:03 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"arena2r")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘purrr’ ‘shinyBS’ ‘shinydashboard’ ‘shinyjs’
      All declared Imports should be used.
    ```

# AUCell

<details>

* Version: 1.4.1
* Source code: ???
* URL: http://scenic.aertslab.org
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 156

Run `revdep_details(,"AUCell")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doMC’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rbokeh’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .cellProps_plotTsne: warning in adjustcolor(colorPal(10), alpha = 0.8):
      partial argument match of 'alpha' to 'alpha.f'
    AUCell_createViewerApp : <anonymous>: no visible global function
      definition for ‘%>%’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘tsne1’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘tsne2’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘cell’
    Undefined global functions or variables:
      %>% cell tsne1 tsne2
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘SingleCellExperiment’
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘reshape2’
    ```

# BatchQC

<details>

* Version: 1.10.1
* Source code: ???
* URL: https://github.com/mani2012/BatchQC
* BugReports: https://github.com/mani2012/BatchQC/issues
* Date/Publication: 2018-12-02
* Number of recursive dependencies: 121

Run `revdep_details(,"BatchQC")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    gene_plot: no visible global function definition for ‘boxplot’
    gene_plot: no visible binding for global variable ‘batch’
    gls.series.C: no visible global function definition for ‘lm.fit’
    sample_plot: no visible global function definition for ‘boxplot’
    sample_plot: no visible binding for global variable ‘batch’
    Undefined global functions or variables:
      batch boxplot lm.fit
    Consider adding
      importFrom("graphics", "boxplot")
      importFrom("stats", "lm.fit")
    to your NAMESPACE file.
    ```

# BayesBD

<details>

* Version: 1.2
* Source code: https://github.com/cran/BayesBD
* Date/Publication: 2017-08-21 17:32:06 UTC
* Number of recursive dependencies: 26

Run `revdep_details(,"BayesBD")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mritc’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BayesianNetwork

<details>

* Version: 0.1.5
* Source code: https://github.com/cran/BayesianNetwork
* URL: https://github.com/paulgovan/bayesiannetwork
* BugReports: https://github.com/paulgovan/bayesiannetwork/issues
* Date/Publication: 2018-12-02 05:10:03 UTC
* Number of recursive dependencies: 121

Run `revdep_details(,"BayesianNetwork")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘shinytest’ ‘testthat’
      All declared Imports should be used.
    ```

# BCEA

<details>

* Version: 2.2-6
* Source code: https://github.com/cran/BCEA
* URL: http://www.statistica.it/gianluca/BCEA, http://www.statistica.it/gianluca
* Date/Publication: 2018-07-11 11:10:03 UTC
* Number of recursive dependencies: 56

Run `revdep_details(,"BCEA")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# bdchecks

<details>

* Version: 0.1.7
* Source code: https://github.com/cran/bdchecks
* URL: https://github.com/bd-R/bdchecks
* BugReports: https://github.com/bd-R/bdchecks/issues
* Date/Publication: 2019-02-18 15:00:07 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"bdchecks")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 72 marked UTF-8 strings
    ```

# bdclean

<details>

* Version: 0.1.15
* Source code: https://github.com/cran/bdclean
* URL: https://github.com/bd-R/bdclean, https://bd-r.github.io/The-bdverse/index.html
* BugReports: https://github.com/bd-R/bdclean/issues
* Date/Publication: 2019-04-11 14:45:17 UTC
* Number of recursive dependencies: 119

Run `revdep_details(,"bdclean")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# bde

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/bde
* Date/Publication: 2015-02-16 13:48:05
* Number of recursive dependencies: 43

Run `revdep_details(,"bde")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    rsample,BoundedDensity: no visible global function definition for
      ‘runif’
    setNormalizationConst,MacroBetaChen99Kernel: no visible global function
      definition for ‘integrate’
    setNormalizationConst,MacroBetaHirukawaTSKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstant,MacroBetaHirukawaJLNKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstants,MicroBetaChen99Kernel :
      density.kernelFunction: no visible global function definition for
      ‘dbeta’
    setNormalizationConstants,MicroBetaChen99Kernel : <anonymous>: no
      visible global function definition for ‘integrate’
    show,BoundedDensity: no visible global function definition for ‘str’
    show,KernelDensity: no visible global function definition for ‘str’
    Undefined global functions or variables:
      dbeta integrate runif str
    Consider adding
      importFrom("stats", "dbeta", "integrate", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# bea.R

<details>

* Version: 1.0.6
* Source code: https://github.com/cran/bea.R
* URL: https://github.com/us-bea/bea.R
* Date/Publication: 2018-02-23 19:30:19 UTC
* Number of recursive dependencies: 56

Run `revdep_details(,"bea.R")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘chron’ ‘colorspace’ ‘gtable’ ‘htmltools’ ‘htmlwidgets’
      ‘httpuv’ ‘magrittr’ ‘munsell’ ‘plyr’ ‘scales’ ‘stringi’ ‘xtable’
      ‘yaml’
      All declared Imports should be used.
    ```

# beanz

<details>

* Version: 2.4
* Source code: https://github.com/cran/beanz
* Date/Publication: 2018-11-05 21:30:03 UTC
* Number of recursive dependencies: 77

Run `revdep_details(,"beanz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        libs   7.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# bigQueryR

<details>

* Version: 0.4.0
* Source code: https://github.com/cran/bigQueryR
* URL: http://code.markedmondson.me/bigQueryR/
* BugReports: https://github.com/cloudyr/bigQueryR/issues
* Date/Publication: 2018-06-08 15:15:43 UTC
* Number of recursive dependencies: 44

Run `revdep_details(,"bigQueryR")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘bigrquery’
    ```

# bioCancer

<details>

* Version: 1.10.11
* Source code: ???
* URL: http://kmezhoud.github.io/bioCancer
* BugReports: https://github.com/kmezhoud/bioCancer/issues
* Date/Publication: 2019-04-04
* Number of recursive dependencies: 189

Run `revdep_details(,"bioCancer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        app   3.3Mb
        doc   2.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘AlgDesign’ ‘dplyr’ ‘import’ ‘methods’ ‘shinythemes’
      All declared Imports should be used.
    ```

# BiocOncoTK

<details>

* Version: 1.2.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 140

Run `revdep_details(,"BiocOncoTK")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘EnsDb.Hsapiens.v75’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   4.0Mb
        doc    1.9Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘DBI’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .rainfall.maeGRL.df: no visible global function definition for
      ‘seqlengths’
    checkCache_patel: no visible global function definition for
      ‘BiocFileCache’
    chrbounds_basic: no visible global function definition for ‘seqlengths’
    ggFeatDens : <anonymous>: no visible binding for global variable
      ‘Consequence’
    ggFeatDens: no visible binding for global variable ‘tfstart’
    ggFeatureSegs: no visible global function definition for ‘genes’
    ggFeatureSegs: no visible binding for global variable ‘symbol’
    ggMutDens : <anonymous>: no visible binding for global variable
      ‘Consequence’
    ggMutDens: no visible binding for global variable ‘project_short_name’
    mc3toGR : <anonymous>: no visible binding for global variable
      ‘Consequence’
    rainfall: no visible global function definition for ‘genome’
    tumNorSet : <anonymous>: no visible global function definition for
      ‘pancan_SE’
    Undefined global functions or variables:
      BiocFileCache Consequence genes genome pancan_SE project_short_name
      seqlengths symbol tfstart
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 46 marked UTF-8 strings
    ```

# BioInstaller

<details>

* Version: 0.3.7
* Source code: https://github.com/cran/BioInstaller
* URL: https://github.com/JhuangLab/BioInstaller
* BugReports: https://github.com/JhuangLab/BioInstaller/issues
* Date/Publication: 2018-11-20 15:50:09 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"BioInstaller")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   8.1Mb
    ```

# BioNetStat

<details>

* Version: 1.2.2
* Source code: ???
* URL: http://github.com/jardimViniciusC/BioNetStat
* BugReports: http://github.com/jardimViniciusC/BioNetStat/issues
* Date/Publication: 2018-12-17
* Number of recursive dependencies: 109

Run `revdep_details(,"BioNetStat")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        extdata   1.9Mb
        shiny     1.6Mb
    ```

# biva

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/biva
* Date/Publication: 2018-10-30 19:20:06 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"biva")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘corrgram’ ‘rpivotTable’ ‘shinyAce’
      All declared Imports should be used.
    ```

# blkbox

<details>

* Version: 1.0
* Source code: https://github.com/cran/blkbox
* Date/Publication: 2016-08-08 02:37:07
* Number of recursive dependencies: 138

Run `revdep_details(,"blkbox")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bigrf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘glmnet’ ‘gtools’ ‘knitr’ ‘nnet’ ‘parallel’ ‘rJava’ ‘reshape’
      ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    Missing or unexported object: ‘xgboost::predict’
    ```

# blockseg

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/blockseg
* URL: https://github.com/jchiquet/blockseg (dev version)
* Date/Publication: 2018-07-03 20:40:03 UTC
* Number of recursive dependencies: 44

Run `revdep_details(,"blockseg")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# bpbounds

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/bpbounds
* URL: https://github.com/remlapmot/bpbounds
* BugReports: https://github.com/remlapmot/bpbounds/issues
* Date/Publication: 2019-02-10 12:13:12 UTC
* Number of recursive dependencies: 43

Run `revdep_details(,"bpbounds")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# bsplus

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/bsplus
* URL: https://github.com/ijlyttle/bsplus
* BugReports: https://github.com/ijlyttle/bsplus/issues
* Date/Publication: 2018-04-05 18:39:46 UTC
* Number of recursive dependencies: 43

Run `revdep_details(,"bsplus")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# caffsim

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/caffsim
* URL: https://github.com/asancpt/caffsim
* BugReports: https://github.com/asancpt/caffsim/issues
* Date/Publication: 2017-08-28 03:36:21 UTC
* Number of recursive dependencies: 49

Run `revdep_details(,"caffsim")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# CaPO4Sim

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CaPO4Sim
* Date/Publication: 2019-04-11 14:55:39 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"CaPO4Sim")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# cartools

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/cartools
* Date/Publication: 2018-08-20 09:00:03 UTC
* Number of recursive dependencies: 118

Run `revdep_details(,"cartools")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘animation’ ‘devtools’ ‘gapminder’ ‘knitr’ ‘rlist’ ‘rmarkdown’
      ‘roxygen2’ ‘sde’ ‘shiny’ ‘tidyverse’ ‘usethis’ ‘utils’
      All declared Imports should be used.
    ```

# CATALYST

<details>

* Version: 1.6.7
* Source code: ???
* URL: https://github.com/HelenaLC/CATALYST
* BugReports: https://github.com/HelenaLC/CATALYST/issues
* Date/Publication: 2019-03-21
* Number of recursive dependencies: 181

Run `revdep_details(,"CATALYST")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.5Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc    5.2Mb
    ```

# ChAMP

<details>

* Version: 2.12.4
* Source code: ???
* Date/Publication: 2019-02-06
* Number of recursive dependencies: 228

Run `revdep_details(,"ChAMP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘IlluminaHumanMethylation450kmanifest’
    
    Depends: includes the non-default packages:
      ‘minfi’ ‘ChAMPdata’ ‘FEM’ ‘DMRcate’ ‘Illumina450ProbeVariants.db’
      ‘IlluminaHumanMethylationEPICmanifest’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ChannelAttributionApp

<details>

* Version: 1.1
* Source code: https://github.com/cran/ChannelAttributionApp
* URL: http://www.slideshare.net/adavide1982/markov-model-for-the-multichannel-attribution-problem
* Date/Publication: 2016-02-21 19:51:20
* Number of recursive dependencies: 46

Run `revdep_details(,"ChannelAttributionApp")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

# chipPCR

<details>

* Version: 0.0.8-10
* Source code: https://github.com/cran/chipPCR
* URL: https://github.com/michbur/chipPCR
* BugReports: https://github.com/michbur/chipPCR/issues
* Date/Publication: 2015-04-10 01:10:56
* Number of recursive dependencies: 119

Run `revdep_details(,"chipPCR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot,refMFI-refMFI: no visible global function definition for ‘mtext’
    plot,refMFI-refMFI: no visible global function definition for ‘abline’
    plot,refMFI-refMFI: no visible global function definition for ‘legend’
    plot,refMFI-refMFI: no visible global function definition for ‘lines’
    qqline,refMFI: no visible binding for global variable ‘qnorm’
    Undefined global functions or variables:
      abline adjustcolor arrows axis coef cor head layout legend lines lm
      mad matplot median mtext na.omit par points polygon predict
      predict.lm qnorm quantile rect rnorm rstudent rug sd shapiro.test
      smooth.spline tail text wilcox.test
    Consider adding
      importFrom("grDevices", "adjustcolor")
      importFrom("graphics", "abline", "arrows", "axis", "layout", "legend",
                 "lines", "matplot", "mtext", "par", "points", "polygon",
                 "rect", "rug", "text")
      importFrom("stats", "coef", "cor", "lm", "mad", "median", "na.omit",
                 "predict", "predict.lm", "qnorm", "quantile", "rnorm",
                 "rstudent", "sd", "shapiro.test", "smooth.spline",
                 "wilcox.test")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

# chromoMap

<details>

* Version: 0.2
* Source code: https://github.com/cran/chromoMap
* Date/Publication: 2019-04-10 19:27:11 UTC
* Number of recursive dependencies: 29

Run `revdep_details(,"chromoMap")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# chromVAR

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 132

Run `revdep_details(,"chromVAR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘chromVAR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: addGCBias
    > ### Title: addGCBias
    > ### Aliases: addGCBias addGCBias,RangedSummarizedExperiment-method
    > ###   addGCBias,SummarizedExperiment-method
    > 
    > ### ** Examples
    > 
    > 
    > data(example_counts, package = "chromVAR")
    > # show example on small part of data 
    > subset_counts <- example_counts[1:500,]
    > library(BSgenome.Hsapiens.UCSC.hg19)
    Error in library(BSgenome.Hsapiens.UCSC.hg19) : 
      there is no package called ‘BSgenome.Hsapiens.UCSC.hg19’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BSgenome.Hsapiens.UCSC.hg19’
    ```

# citr

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/citr
* URL: https://github.com/crsh/citr
* BugReports: https://github.com/crsh/citr/issues
* Date/Publication: 2018-12-18 20:00:03 UTC
* Number of recursive dependencies: 38

Run `revdep_details(,"citr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘bibtex’
      All declared Imports should be used.
    ```

# cjoint

<details>

* Version: 2.1.0
* Source code: https://github.com/cran/cjoint
* Date/Publication: 2018-10-19 08:20:02 UTC
* Number of recursive dependencies: 58

Run `revdep_details(,"cjoint")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyBS’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 29 marked UTF-8 strings
    ```

# clustDRM

<details>

* Version: 0.1-0
* Source code: https://github.com/cran/clustDRM
* Date/Publication: 2019-03-15 16:53:44 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"clustDRM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘IsoGene’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CNVPanelizer

<details>

* Version: 1.14.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 86

Run `revdep_details(,"CNVPanelizer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘utils:::format.object_size’
      See the note in ?`:::` about the use of this operator.
    ```

# CNVScope

<details>

* Version: 1.9.7
* Source code: https://github.com/cran/CNVScope
* URL: https://github.com/jamesdalg/CNVScope/
* BugReports: https://github.com/jamesdalg/CNVScope/issues/
* Date/Publication: 2018-10-20 07:10:08 UTC
* Number of recursive dependencies: 197

Run `revdep_details(,"CNVScope")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘BSgenome.Hsapiens.UCSC.hg19’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# cocktailApp

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/cocktailApp
* URL: https://github.com/shabbychef/cocktailApp
* BugReports: https://github.com/shabbychef/cocktailApp/issues
* Date/Publication: 2018-08-19 16:40:02 UTC
* Number of recursive dependencies: 58

Run `revdep_details(,"cocktailApp")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14661 marked UTF-8 strings
    ```

# codebook

<details>

* Version: 0.8.0
* Source code: https://github.com/cran/codebook
* URL: https://github.com/rubenarslan/codebook
* BugReports: https://github.com/rubenarslan/codebook/issues
* Date/Publication: 2019-02-21 15:20:07 UTC
* Number of recursive dependencies: 169

Run `revdep_details(,"codebook")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘jsonlite’ ‘pander’ ‘rlang’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘mice’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 65 marked UTF-8 strings
    ```

# compareGroups

<details>

* Version: 4.0.0
* Source code: https://github.com/cran/compareGroups
* URL: http://www.comparegroups.eu
* Date/Publication: 2018-11-02 14:50:06 UTC
* Number of recursive dependencies: 135

Run `revdep_details(,"compareGroups")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        app   2.7Mb
        doc   2.3Mb
    ```

# COMPASS

<details>

* Version: 1.20.1
* Source code: ???
* BugReports: https://github.com/RGLab/COMPASS/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 119

Run `revdep_details(,"COMPASS")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocStyle’ ‘rmarkdown’
      All declared Imports should be used.
    ':::' call which should be '::': ‘flowWorkspace:::.getNodeInd’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .CellCounts: no visible binding for global variable
      ‘_COMPASS_CellCounts’
    COMPASSfitToCountsTable: no visible binding for global variable
      ‘population’
    COMPASSfitToCountsTable: no visible binding for global variable ‘Count’
    COMPASSfitToCountsTable: no visible binding for global variable ‘id’
    CellCounts_character: no visible binding for global variable
      ‘_COMPASS_CellCounts_character’
    Undefined global functions or variables:
      Count _COMPASS_CellCounts _COMPASS_CellCounts_character id population
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘ggplot2’ ‘readxl’
    ```

# condformat

<details>

* Version: 0.8.0
* Source code: https://github.com/cran/condformat
* URL: http://github.com/zeehio/condformat
* BugReports: http://github.com/zeehio/condformat/issues
* Date/Publication: 2018-10-29 14:10:03 UTC
* Number of recursive dependencies: 59

Run `revdep_details(,"condformat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    ```

# CoRegNet

<details>

* Version: 1.20.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 43

Run `revdep_details(,"CoRegNet")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      Loading required package: arules
      Loading required package: Matrix
      
      Attaching package: 'arules'
      
      The following objects are masked from 'package:base':
      
          abbreviate, write
      
      [1] TRUE
      > BiocGenerics:::testPackage('CoRegNet')
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking for unstated dependencies in ‘tests’ ... WARNING
    ```
    '::' or ':::' import not declared from: ‘BiocGenerics’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘RColorBrewer’ ‘gplots’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    refine,coregnet: no visible global function definition for ‘abline’
    regulatorInfluence,coregnet : <anonymous> : <anonymous>: no visible
      global function definition for ‘t.test’
    regulators,coregnet: no visible global function definition for
      ‘na.omit’
    repressors,coregnet: no visible global function definition for
      ‘na.omit’
    targets,coregnet: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      abline arrows as.dist axis coef colorRampPalette cor fisher.test glm
      hclust heatmap.2 lines lm mtext na.omit p.adjust par pchisq plot
      pnorm quantile rainbow sd segments symbols t.test text vcount
      wilcox.test
    Consider adding
      importFrom("grDevices", "colorRampPalette", "rainbow")
      importFrom("graphics", "abline", "arrows", "axis", "lines", "mtext",
                 "par", "plot", "segments", "symbols", "text")
      importFrom("stats", "as.dist", "coef", "cor", "fisher.test", "glm",
                 "hclust", "lm", "na.omit", "p.adjust", "pchisq", "pnorm",
                 "quantile", "sd", "t.test", "wilcox.test")
    to your NAMESPACE file.
    ```

# cosinor

<details>

* Version: 1.1
* Source code: https://github.com/cran/cosinor
* URL: http://github.com/sachsmc/cosinor
* Date/Publication: 2014-07-28 20:04:18
* Number of recursive dependencies: 49

Run `revdep_details(,"cosinor")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    cosinor_analyzer : <anonymous>: no visible global function definition
      for ‘as.formula’
    ggplot.cosinor.lm: no visible global function definition for ‘predict’
    predict.cosinor.lm: no visible global function definition for ‘fitted’
    predict.cosinor.lm: no visible global function definition for ‘predict’
    simulate_cosinor: no visible global function definition for ‘runif’
    simulate_cosinor: no visible global function definition for ‘rbinom’
    simulate_cosinor: no visible global function definition for ‘rnorm’
    summary.cosinor.lm: no visible global function definition for ‘vcov’
    summary.cosinor.lm: no visible global function definition for ‘qnorm’
    summary.cosinor.lm: no visible global function definition for ‘pnorm’
    test_cosinor: no visible global function definition for ‘pchisq’
    test_cosinor: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      as.formula fitted lm na.omit pchisq pnorm predict qnorm rbinom rnorm
      runif terms vcov vitamind
    Consider adding
      importFrom("stats", "as.formula", "fitted", "lm", "na.omit", "pchisq",
                 "pnorm", "predict", "qnorm", "rbinom", "rnorm", "runif",
                 "terms", "vcov")
    to your NAMESPACE file.
    ```

# coveffectsplot

<details>

* Version: 0.0.3
* Source code: https://github.com/cran/coveffectsplot
* URL: https://github.com/smouksassi/interactiveforestplot
* BugReports: https://github.com/smouksassi/interactiveforestplot/issues
* Date/Publication: 2019-02-25 10:30:04 UTC
* Number of recursive dependencies: 74

Run `revdep_details(,"coveffectsplot")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘dplyr’ ‘markdown’ ‘shinyjs’ ‘tidyr’
      All declared Imports should be used.
    ```

# CRANsearcher

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/CRANsearcher
* URL: https://github.com/RhoInc/CRANsearcher
* BugReports: https://github.com/RhoInc/CRANsearcher/issues
* Date/Publication: 2017-06-26 23:13:21 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"CRANsearcher")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 11 marked Latin-1 strings
      Note: found 57 marked UTF-8 strings
    ```

# crawl

<details>

* Version: 2.2.1
* Source code: https://github.com/cran/crawl
* Date/Publication: 2018-09-14 19:20:10 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"crawl")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gdistance’ ‘raster’
      All declared Imports should be used.
    ```

# crisprseekplus

<details>

* Version: 1.8.0
* Source code: ???
* URL: https://github.com/UMMS-Biocore/crisprseekplus
* BugReports: https://github.com/UMMS-Biocore/crisprseekplus/issues/new
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 128

Run `revdep_details(,"crisprseekplus")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    installpack: no visible global function definition for
      'install.packages'
    Undefined global functions or variables:
      install.packages
    Consider adding
      importFrom("utils", "install.packages")
    to your NAMESPACE file.
    ```

# crossmeta

<details>

* Version: 1.8.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 160

Run `revdep_details(,"crossmeta")` for more info

</details>

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Data with usage in documentation object 'gs.names' but not in code:
      gs.names
    
    Data with usage in documentation object 'gslist' but not in code:
      gslist
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported objects:
      ‘crossmeta::gs.names’ ‘crossmeta::gslist’
    ```

*   checking R code for possible problems ... NOTE
    ```
    explore_paths : server: no visible binding for global variable ‘gslist’
    explore_paths : server: no visible binding for global variable
      ‘gs.names’
    Undefined global functions or variables:
      gs.names gslist
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘PADOG’, ‘GeneMeta’
    ```

# CTAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CTAShiny
* Date/Publication: 2019-04-13 08:22:55 UTC
* Number of recursive dependencies: 21

Run `revdep_details(,"CTAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘epitools’ ‘rpivotTable’ ‘shinyMatrix’
      All declared Imports should be used.
    ```

# CTRE

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CTRE
* URL: https://unsw-math.github.io/CTRE/
* BugReports: https://github.com/UNSW-MATH/CTRE/issues
* Date/Publication: 2018-05-07 13:30:03 UTC
* Number of recursive dependencies: 53

Run `revdep_details(,"CTRE")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tea’
      All declared Imports should be used.
    ```

# ctsem

<details>

* Version: 2.9.0
* Source code: https://github.com/cran/ctsem
* URL: https://github.com/cdriveraus/ctsem
* Date/Publication: 2019-04-13 22:52:46 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"ctsem")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        libs   5.8Mb
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# CVE

<details>

* Version: 1.8.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 178

Run `revdep_details(,"CVE")` for more info

</details>

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                                    old_size new_size compress
      WGCNAmelanoma_extension.RData    2.4Mb    2.1Mb       xz
      crcCase.RData                    1.1Mb    677Kb       xz
      melanomaCase.RData               654Kb    472Kb       xz
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   4.5Mb
        doc    1.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘tidyverse’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# cydar

<details>

* Version: 1.6.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 124

Run `revdep_details(,"cydar")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# cytofkit

<details>

* Version: 1.12.0
* Source code: ???
* URL: https://github.com/JinmiaoChenLab/cytofkit
* BugReports: https://github.com/JinmiaoChenLab/cytofkit/issues
* Number of recursive dependencies: 158

Run `revdep_details(,"cytofkit")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > m1 <- c(rnorm(300, 10, 2), rnorm(400, 4, 2), rnorm(300, 7))
    > m2 <- c(rnorm(300, 4), rnorm(400, 16), rnorm(300, 10, 3))
    > m3 <- c(rnorm(300, 16), rnorm(400, 40, 3), rnorm(300, 10))
    > m4 <- c(rnorm(300, 7, 3), rnorm(400, 30, 2), rnorm(300, 10))
    > m5 <- c(rnorm(300, 27), rnorm(400, 40, 1),rnorm(300, 10))
    > c <- c(rep(1,300), rep(2,400), rep(3,300))
    > rnames <- paste(paste('sample_', c('A','B','C','D'), sep = ''), 
    + rep(1:250,each = 4), sep='_') 
    > exprs_cluster <- data.frame(cluster = c, m1 = m1, m2 = m2, m3 = m3, m4 = m4, isomap_1 = m5)
    > row.names(exprs_cluster) <- sample(rnames, 1000)
    > cytof_progressionPlot(exprs_cluster, markers = c("m1","m2","m3","m4"))
    [1] "Error!"
    <simpleError in eval(call[[1L]]): object 'sm.ns' not found>
    [1] "Error!"
    <simpleError in eval(call[[1L]]): object 'sm.ns' not found>
    [1] "Error!"
    <simpleError in eval(call[[1L]]): object 'sm.ns' not found>
    [1] "Error!"
    <simpleError in eval(call[[1L]]): object 'sm.ns' not found>
    Error: Discrete value supplied to continuous scale
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        doc       4.3Mb
        extdata   3.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    cytof_clusterPlot: no visible global function definition for
      ‘geom_text’
    cytof_progressionPlot: no visible global function definition for ‘aes’
    cytof_progressionPlot: no visible binding for global variable
      ‘Pseudotime’
    cytof_progressionPlot: no visible binding for global variable ‘cluster’
    cytofkitShinyAPP : <anonymous> : C_ScatterPlotInput: no visible global
      function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous>: no visible global function definition
      for ‘heatMap’
    cytofkitShinyAPP : <anonymous> : M_markerExpressionPlotInput: no
      visible global function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous> : M_stackDensityPlotInput: no visible
      global function definition for ‘stackDenistyPlot’
    cytofkitShinyAPP : <anonymous> : P_markerPlotInput: no visible global
      function definition for ‘cytof_expressionTrends’
    Undefined global functions or variables:
      Pseudotime aes cluster cytof_expressionTrends geom_text heatMap
      scatterPlot stackDenistyPlot
    ```

# d3Tree

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/d3Tree
* URL: https://github.com/metrumresearchgroup/d3Tree
* BugReports: https://github.com/metrumresearchgroup/d3Tree/issues
* Date/Publication: 2017-06-13 20:35:10 UTC
* Number of recursive dependencies: 43

Run `revdep_details(,"d3Tree")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# datacheck

<details>

* Version: 1.2.2
* Source code: https://github.com/cran/datacheck
* URL: https://github.com/c5sire/datacheck
* BugReports: https://github.com/c5sire/datacheck/issues
* Date/Publication: 2015-04-18 08:56:10
* Number of recursive dependencies: 67

Run `revdep_details(,"datacheck")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    is_one_of: no visible global function definition for ‘read.csv’
    pkg.version: no visible global function definition for ‘citation’
    pkg_version: no visible global function definition for ‘citation’
    ruleCoverage: no visible global function definition for ‘dotchart’
    ruleCoverage: no visible global function definition for ‘abline’
    rule_coverage: no visible global function definition for ‘dotchart’
    rule_coverage: no visible global function definition for ‘abline’
    scoreSum: no visible global function definition for ‘plot’
    score_sum: no visible global function definition for ‘plot’
    shortSummary: no visible global function definition for ‘sd’
    short_summary: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      abline citation colorRampPalette dotchart heatmap plot read.csv sd
      setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "dotchart", "plot")
      importFrom("stats", "heatmap", "sd")
      importFrom("utils", "citation", "read.csv", "setTxtProgressBar",
                 "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

# datadigest

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/datadigest
* URL: https://github.com/RhoInc/datadigest
* BugReports: https://github.com/RhoInc/datadigest/issues
* Date/Publication: 2018-09-03 11:10:06 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"datadigest")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘miniUI’
      All declared Imports should be used.
    ```

# debrowser

<details>

* Version: 1.10.9
* Source code: ???
* URL: https://github.com/UMMS-Biocore/debrowser
* BugReports: https://github.com/UMMS-Biocore/debrowser/issues/new
* Date/Publication: 2019-02-03
* Number of recursive dependencies: 205

Run `revdep_details(,"debrowser")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    checkCountData : <anonymous>: no visible binding for global variable
      ‘err’
    debrowserdataload: no visible binding for global variable ‘demodata’
    getDensityPlot: no visible binding for global variable ‘samples’
    getMean: no visible binding for global variable ‘norm_data’
    plot_pca: no visible binding for global variable ‘x’
    plot_pca: no visible binding for global variable ‘y’
    plot_pca: no visible binding for global variable ‘samples’
    Undefined global functions or variables:
      demodata err norm_data samples x y
    ```

# DEP

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 131

Run `revdep_details(,"DEP")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        doc    3.1Mb
    ```

# detzrcr

<details>

* Version: 0.2.5
* Source code: https://github.com/cran/detzrcr
* URL: https://github.com/magnuskristoffersen/detzrcr
* Date/Publication: 2019-03-27 11:00:03 UTC
* Number of recursive dependencies: 52

Run `revdep_details(,"detzrcr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# dextergui

<details>

* Version: 0.1.6
* Source code: https://github.com/cran/dextergui
* BugReports: https://github.com/jessekps/dexter/issues
* Date/Publication: 2019-03-06 13:00:03 UTC
* Number of recursive dependencies: 118

Run `revdep_details(,"dextergui")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘dexter:::get_resp_data’ ‘dexter:::qcolors’
      See the note in ?`:::` about the use of this operator.
    ```

# DLMtool

<details>

* Version: 5.3
* Source code: https://github.com/cran/DLMtool
* URL: http://www.datalimitedtoolkit.org/
* BugReports: https://github.com/DLMtool/DLMtool/issues
* Date/Publication: 2019-03-06 22:13:08 UTC
* Number of recursive dependencies: 116

Run `revdep_details(,"DLMtool")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.1Mb
    ```

# Doscheda

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 152

Run `revdep_details(,"Doscheda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   2.0Mb
    ```

# dplyrAssist

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/dplyrAssist
* URL: https://github.com/cardiomoon/dplyrAssist
* BugReports: https://github.com/cardiomoon/dplyrAssist/issues
* Date/Publication: 2017-09-04 17:46:22 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"dplyrAssist")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘tidyr’ ‘tidyverse’
      All declared Imports should be used.
    ```

# dropR

<details>

* Version: 0.1
* Source code: https://github.com/cran/dropR
* Date/Publication: 2015-01-02 06:52:56
* Number of recursive dependencies: 17

Run `revdep_details(,"dropR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    compute_xsq: no visible global function definition for ‘chisq.test’
    Undefined global functions or variables:
      chisq.test
    Consider adding
      importFrom("stats", "chisq.test")
    to your NAMESPACE file.
    ```

# dtwclust

<details>

* Version: 5.5.2
* Source code: https://github.com/cran/dtwclust
* URL: https://github.com/asardaes/dtwclust
* BugReports: https://github.com/asardaes/dtwclust/issues
* Date/Publication: 2019-03-09 15:12:40 UTC
* Number of recursive dependencies: 75

Run `revdep_details(,"dtwclust")` for more info

</details>

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# dynamichazard

<details>

* Version: 0.6.5
* Source code: https://github.com/cran/dynamichazard
* URL: https://github.com/boennecd/dynamichazard
* BugReports: https://github.com/boennecd/dynamichazard/issues
* Date/Publication: 2019-04-05 07:10:03 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"dynamichazard")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
        doc    1.6Mb
        libs   2.6Mb
    ```

# EBImage

<details>

* Version: 4.24.0
* Source code: ???
* URL: https://github.com/aoles/EBImage
* BugReports: https://github.com/aoles/EBImage/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 42

Run `revdep_details(,"EBImage")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.4Mb
      sub-directories of 1Mb or more:
        doc     14.0Mb
        images   1.7Mb
    ```

# effectR

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/effectR
* Date/Publication: 2018-09-30 21:30:03 UTC
* Number of recursive dependencies: 59

Run `revdep_details(,"effectR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rmarkdown’
      All declared Imports should be used.
    ```

# EHRtemporalVariability

<details>

* Version: 1.0
* Source code: https://github.com/cran/EHRtemporalVariability
* URL: http://github.com/hms-dbmi/EHRtemporalVariability
* Date/Publication: 2019-03-14 12:03:34 UTC
* Number of recursive dependencies: 99

Run `revdep_details(,"EHRtemporalVariability")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.2Mb
      sub-directories of 1Mb or more:
        doc       4.9Mb
        extdata   5.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘shiny’
      All declared Imports should be used.
    ```

# elementR

<details>

* Version: 1.3.6
* Source code: https://github.com/cran/elementR
* URL: https://github.com/charlottesirot/elementR
* BugReports: https://github.com/charlottesirot/elementR/issues
* Date/Publication: 2018-05-03 14:08:24 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"elementR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘elementR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: elementR_project
    > ### Title: Object elementR_project
    > ### Aliases: elementR_project
    > 
    > ### ** Examples
    > 
    > ## create a new elementR_repStandard object based on the "filePath" 
    > ## from a folder containing sample replicate
    > 
    > filePath <- system.file("Example_Session", package="elementR")
    > 
    > exampleProject <- elementR_project$new(filePath)
    Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
      [tcl] invalid command name "toplevel".
    Calls: <Anonymous> ... tktoplevel -> tkwidget -> tcl -> .Tcl.objv -> structure
    Execution halted
    ```

# embryogrowth

<details>

* Version: 7.4.1
* Source code: https://github.com/cran/embryogrowth
* Date/Publication: 2018-09-27 12:00:03 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"embryogrowth")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5 marked UTF-8 strings
    ```

# EmiStatR

<details>

* Version: 1.2.1.2
* Source code: https://github.com/cran/EmiStatR
* Date/Publication: 2019-03-01 07:07:04 UTC
* Number of recursive dependencies: 23

Run `revdep_details(,"EmiStatR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    The following objects are masked from ‘package:base’:
    
        as.Date, as.Date.numeric
    
    > 
    > data("Esch_Sure2010")
    > 
    > cinp            <- 150 # water consumption [m3/h]
    > prec            <- Esch_Sure2010[1:1000,] # selecting just the first 1,000 rows
    > cinp.daily.dir  <- system.file("shiny/EmiStatR_inputCSO/inputCSO", package = "EmiStatR")
    > cinp.daily.file <- paste(cinp.daily.dir, "/qs_factor.csv", sep="")
    > cinp.weekly     <- list(mon=1, tue=.83, wed=.83, thu=.83, fri=1, sat=1.25, sun=1.25)  
    > # factors average to 1
    > cinp.seasonal   <- list(jan=.79, feb=.79, mar=1.15, apr=1.15, may=1.15, jun=1.15,
    +                         jul=1.15, aug=1.15, sep=1.15, oct=1.15, nov=.79, dec=.79) 
    > # factors average to 1
    > 
    > ts1 <- CInp2TS(cinp, prec, cinp.daily.file, cinp.weekly, cinp.seasonal)
    Error in `[.xts`(cinp.ts.deltap, 1:nrow(P1), ) : subscript out of bounds
    Calls: CInp2TS -> [ -> [.xts
    Execution halted
    ```

# emuR

<details>

* Version: 1.1.2
* Source code: https://github.com/cran/emuR
* URL: https://github.com/IPS-LMU/emuR
* BugReports: https://github.com/IPS-LMU/emuR/issues
* Date/Publication: 2019-01-29 13:20:03 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"emuR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘git2r’ ‘servr’
      All declared Imports should be used.
    ```

# ENCODExplorer

<details>

* Version: 2.8.0
* Source code: ???
* BugReports: https://github.com/CharlesJB/ENCODExplorer/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 68

Run `revdep_details(,"ENCODExplorer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 73.8Mb
      sub-directories of 1Mb or more:
        data     24.1Mb
        doc       1.5Mb
        extdata  48.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘biosample_type’
    step6_control: no visible binding for global variable ‘controls’
    step6_date_released: no visible binding for global variable
      ‘date_released’
    step6_status: no visible binding for global variable ‘status’
    step6_target: no visible binding for global variable ‘target’
    step7: no visible binding for global variable ‘organism’
    step8: no visible binding for global variable ‘investigated_as’
    step8: no visible binding for global variable ‘target’
    step9: no visible binding for global variable ‘organism’
    Undefined global functions or variables:
      . Experiment Value accession antibody_caption
      antibody_characterization antibody_target assay
      biological_replicate_number biosample_name biosample_type col_name
      controls data date_released download.file encode_df file_accession
      file_format href investigated_as lab nucleic_acid_term organism
      platform project replicate_antibody replicate_library server status
      submitted_by target technical_replicate_number treatment ui value
    Consider adding
      importFrom("utils", "data", "download.file")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 771 marked UTF-8 strings
    ```

# enviGCMS

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/enviGCMS
* URL: https://github.com/yufree/enviGCMS
* Date/Publication: 2018-04-25 22:53:04 UTC
* Number of recursive dependencies: 124

Run `revdep_details(,"enviGCMS")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocParallel’ ‘broom’ ‘reshape2’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# epicontacts

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/epicontacts
* URL: http://www.repidemicsconsortium.org/epicontacts/
* BugReports: http://github.com/reconhub/epicontacts/issues
* Date/Publication: 2017-11-21 08:24:53 UTC
* Number of recursive dependencies: 74

Run `revdep_details(,"epicontacts")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘colorspace’
      All declared Imports should be used.
    ```

# EpiSignalDetection

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/EpiSignalDetection
* URL: http://atlas.ecdc.europa.eu/public/index.aspx
* Date/Publication: 2018-11-15 22:00:04 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"EpiSignalDetection")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ggplot2’ ‘knitr’ ‘pander’
      All declared Imports should be used.
    ```

# epivizrChart

<details>

* Version: 1.4.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 161

Run `revdep_details(,"epivizrChart")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘epivizrChart-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot,EpivizEnvironment,ANY-method
    > ### Title: Generic methods to plot charts and add navigation regions
    > ### Aliases: plot,EpivizEnvironment,ANY-method
    > 
    > ### ** Examples
    > 
    > library(epivizrChart)
    > library(Homo.sapiens)
    Error in library(Homo.sapiens) : 
      there is no package called 'Homo.sapiens'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘Mus.musculus’ ‘Homo.sapiens’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 25.1Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc   17.3Mb
        www    3.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'epivizrServer'
      All declared Imports should be used.
    ```

# erma

<details>

* Version: 0.14.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 116

Run `revdep_details(,"erma")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Homo.sapiens’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EventStudy

<details>

* Version: 0.36
* Source code: https://github.com/cran/EventStudy
* URL: http://eventstudytools.com
* BugReports: https://github.com/EventStudyTools/api-wrapper.r/issues
* Date/Publication: 2019-03-14 12:13:31 UTC
* Number of recursive dependencies: 130

Run `revdep_details(,"EventStudy")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc   5.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘curl’ ‘openxlsx’ ‘stringr’
      All declared Imports should be used.
    ```

# fanplot

<details>

* Version: 3.4.1
* Source code: https://github.com/cran/fanplot
* URL: http://gjabel.wordpress.com/category/r/fanplot/
* Date/Publication: 2015-10-07 22:43:43
* Number of recursive dependencies: 17

Run `revdep_details(,"fanplot")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tsbugs’
    ```

# FastqCleaner

<details>

* Version: 1.0.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 87

Run `revdep_details(,"FastqCleaner")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’
    A package should be listed in only one of these fields.
    ```

# fingertipsR

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/fingertipsR
* URL: https://fingertips.phe.org.uk, https://github.com/ropensci/fingertipsR
* BugReports: https://github.com/ropensci/fingertipsR/issues
* Date/Publication: 2019-04-08 21:23:21 UTC
* Number of recursive dependencies: 72

Run `revdep_details(,"fingertipsR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# fitteR

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/fitteR
* Date/Publication: 2017-12-11 14:27:45 UTC
* Number of recursive dependencies: 185

Run `revdep_details(,"fitteR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ExtDist’
    ```

# fitur

<details>

* Version: 0.6.1
* Source code: https://github.com/cran/fitur
* URL: https://github.com/tomroh/fitur
* BugReports: https://github.com/tomroh/fitur/issues
* Date/Publication: 2018-09-01 19:20:03 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"fitur")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstudioapi’
      All declared Imports should be used.
    ```

# flora

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/flora
* URL: http://www.github.com/gustavobio/flora
* BugReports: http://www.github.com/gustavobio/flora/issues
* Date/Publication: 2017-12-04 19:50:03 UTC
* Number of recursive dependencies: 37

Run `revdep_details(,"flora")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        R   7.1Mb
    ```

# flowAI

<details>

* Version: 1.12.7
* Source code: ???
* Date/Publication: 2019-03-04
* Number of recursive dependencies: 68

Run `revdep_details(,"flowAI")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# flowcatchR

<details>

* Version: 1.16.0
* Source code: ???
* URL: https://github.com/federicomarini/flowcatchR, https://federicomarini.github.io/flowcatchR/
* BugReports: https://github.com/federicomarini/flowcatchR/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 87

Run `revdep_details(,"flowcatchR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.6Mb
      sub-directories of 1Mb or more:
        data   1.9Mb
        doc    9.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘EBImage’ which was already attached by Depends.
      Please remove these calls from your code.
    ```

# FunChIP

<details>

* Version: 1.8.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 47

Run `revdep_details(,"FunChIP")` for more info

</details>

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                old_size new_size compress
      peaks.rda    638Kb    112Kb       xz
    ```

*   checking running R code from vignettes ...
    ```
       ‘FunChIP.Rnw’ ... failed
     WARNING
    Errors in running code in vignettes:
    when running code in ‘FunChIP.Rnw’
      ...
    
    > BiocStyle::latex()
    
      When sourcing ‘FunChIP.R’:
    Error: there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 24.4Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata  21.5Mb
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocStyle’
    ```

# g3viz

<details>

* Version: 1.0.4
* Source code: https://github.com/cran/g3viz
* URL: https://github.com/G3viz/g3viz
* BugReports: https://github.com/G3viz/g3viz/issues
* Date/Publication: 2019-04-06 06:12:47 UTC
* Number of recursive dependencies: 62

Run `revdep_details(,"g3viz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.4Mb
      sub-directories of 1Mb or more:
        doc   9.3Mb
    ```

# GA4GHshiny

<details>

* Version: 1.4.0
* Source code: ???
* URL: https://github.com/labbcb/GA4GHshiny
* BugReports: https://github.com/labbcb/GA4GHshiny/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 116

Run `revdep_details(,"GA4GHshiny")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 4 SKIPPED: 0 FAILED: 12
      1. Error: app works (@test-app.R#5) 
      2. Error: initializeReferences works (@test-initializeReferences.R#6) 
      3. Error: initializeVariantSet works (@test-initializeVariantSet.R#6) 
      4. Error: searchVariantsByGeneSymbol works (@test-searchVariantsByGeneSymbol.R#9) 
      5. Error: searchVariantsByGeneSymbol feature=transcripts works (@test-searchVariantsByGeneSymbol.R#22) 
      6. Error: searchVariantsByGeneSymbol feature=exons works (@test-searchVariantsByGeneSymbol.R#35) 
      7. Error: searchVariantsByGeneSymbol feature=cds works (@test-searchVariantsByGeneSymbol.R#48) 
      8. Error: searchVariantsByGeneSymbol feature=promoters works (@test-searchVariantsByGeneSymbol.R#60) 
      9. Error: searchVariantsByGeneSymbol multiple genes works (@test-searchVariantsByGeneSymbol.R#72) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# gamesGA

<details>

* Version: 1.1.3.6
* Source code: https://github.com/cran/gamesGA
* URL: https://bradduthie.github.io/gamesGA/
* BugReports: https://github.com/bradduthie/gamesGA/issues
* Date/Publication: 2019-03-12 18:12:44 UTC
* Number of recursive dependencies: 21

Run `revdep_details(,"gamesGA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# gastempt

<details>

* Version: 0.4.4
* Source code: https://github.com/cran/gastempt
* URL: http://github.com/dmenne/gastempt
* BugReports: http://github.com/dmenne/gastempt/issues
* Date/Publication: 2019-03-06 16:32:41 UTC
* Number of recursive dependencies: 82

Run `revdep_details(,"gastempt")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        libs   7.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘methods’ ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# GDCRNATools

<details>

* Version: 1.2.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 181

Run `revdep_details(,"GDCRNATools")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        R      3.2Mb
        data   1.1Mb
        doc    2.6Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    gdcRNAMerge: no visible global function definition for ‘read.table’
    gdcRNAMerge : <anonymous>: no visible global function definition for
      ‘read.delim’
    gdcRNAMerge: no visible global function definition for ‘read.delim’
    hyperTestFun: no visible global function definition for ‘phyper’
    kmTestFun: no visible global function definition for ‘pchisq’
    kmTestFun: no visible global function definition for ‘qnorm’
    manifestDownloadFun: no visible global function definition for
      ‘read.table’
    mirCorTestFun: no visible global function definition for ‘cor.test’
    multiRegFun: no visible global function definition for ‘cor.test’
    Undefined global functions or variables:
      Category Counts FDR Regulation Terms URLencode aggregate cor.test
      download.file foldEnrichment model.matrix p.adjust pchisq phyper
      qnorm read.delim read.table unzip write.table
    Consider adding
      importFrom("stats", "aggregate", "cor.test", "model.matrix",
                 "p.adjust", "pchisq", "phyper", "qnorm")
      importFrom("utils", "URLencode", "download.file", "read.delim",
                 "read.table", "unzip", "write.table")
    to your NAMESPACE file.
    ```

# genBart

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/genBart
* Date/Publication: 2018-03-13 22:13:14 UTC
* Number of recursive dependencies: 145

Run `revdep_details(,"genBart")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘NMF’ ‘RColorBrewer’ ‘VennDiagram’ ‘clValid’ ‘data.table’ ‘grid’
      ‘gtools’ ‘pca3d’ ‘rmarkdown’ ‘scales’ ‘shinydashboard’ ‘shinyjs’
      ‘statmod’ ‘stringr’
      All declared Imports should be used.
    ```

# GeneNetworkBuilder

<details>

* Version: 1.24.1
* Source code: ???
* Date/Publication: 2019-04-09
* Number of recursive dependencies: 64

Run `revdep_details(,"GeneNetworkBuilder")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.4Mb
      sub-directories of 1Mb or more:
        data          2.8Mb
        doc           8.4Mb
        htmlwidgets   1.8Mb
    ```

# GerminaR

<details>

* Version: 1.3
* Source code: https://github.com/cran/GerminaR
* URL: https://flavjack.shinyapps.io/germinaquant/
* BugReports: https://github.com/Flavjack/GerminaR/issues
* Date/Publication: 2019-04-18 16:30:21 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"GerminaR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# ggedit

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/ggedit
* URL: https://github.com/metrumresearchgroup/ggedit
* BugReports: https://github.com/metrumresearchgroup/ggedit/issues
* Date/Publication: 2018-07-03 21:50:02 UTC
* Number of recursive dependencies: 65

Run `revdep_details(,"ggedit")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# ggExtra

<details>

* Version: 0.8
* Source code: https://github.com/cran/ggExtra
* URL: https://github.com/daattali/ggExtra
* BugReports: https://github.com/daattali/ggExtra/issues
* Date/Publication: 2018-04-04 11:11:01 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"ggExtra")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# ggplotAssist

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/ggplotAssist
* URL: https://github.com/cardiomoon/ggplotAssist
* BugReports: https://github.com/cardiomoon/ggplotAssist/issues
* Date/Publication: 2017-11-12 12:24:46 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"ggplotAssist")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gcookbook’ ‘ggthemes’ ‘moonBook’ ‘tidyverse’
      All declared Imports should be used.
    ```

# ggquickeda

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/ggquickeda
* URL: https://github.com/smouksassi/ggquickeda
* BugReports: https://github.com/smouksassi/ggquickeda/issues
* Date/Publication: 2019-04-10 12:02:41 UTC
* Number of recursive dependencies: 117

Run `revdep_details(,"ggquickeda")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Formula’ ‘GGally’ ‘Hmisc’ ‘colourpicker’ ‘dplyr’ ‘ggpmisc’
      ‘ggpubr’ ‘ggrepel’ ‘ggstance’ ‘grDevices’ ‘gridExtra’ ‘lazyeval’
      ‘markdown’ ‘plotly’ ‘quantreg’ ‘rlang’ ‘shinyjs’ ‘survminer’ ‘table1’
      ‘tidyr’
      All declared Imports should be used.
    ```

# googleAuthR

<details>

* Version: 0.7.0
* Source code: https://github.com/cran/googleAuthR
* URL: http://code.markedmondson.me/googleAuthR/
* BugReports: https://github.com/MarkEdmondson1234/googleAuthR/issues
* Date/Publication: 2018-11-16 21:00:03 UTC
* Number of recursive dependencies: 86

Run `revdep_details(,"googleAuthR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# gQTLstats

<details>

* Version: 1.14.1
* Source code: ???
* Date/Publication: 2019-03-20
* Number of recursive dependencies: 189

Run `revdep_details(,"gQTLstats")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Homo.sapiens’
    
    Package suggested but not available for checking: ‘geuvStore2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# gridsampler

<details>

* Version: 0.6
* Source code: https://github.com/cran/gridsampler
* URL: https://github.com/markheckmann/gridsampler
* BugReports: https://github.com/markheckmann/gridsampler/issues
* Date/Publication: 2016-11-23 17:24:13
* Number of recursive dependencies: 57

Run `revdep_details(,"gridsampler")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiasedUrn’
      All declared Imports should be used.
    ```

# GSCA

<details>

* Version: 2.12.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 56

Run `revdep_details(,"GSCA")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘shiny’ ‘sp’ ‘gplots’ ‘ggplot2’ ‘reshape2’ ‘RColorBrewer’ ‘rhdf5’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Affyhgu133A2Expr’ ‘Affyhgu133Plus2Expr’ ‘Affyhgu133aExpr’
      ‘Affymoe4302Expr’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    GSCAplot: no visible global function definition for ‘title’
    GSCAplot: no visible global function definition for ‘dev.off’
    annotatePeaks: no visible binding for global variable ‘allreffile’
    tabSearch: no visible global function definition for ‘data’
    tabSearch: no visible binding for global variable ‘Affyhgu133aExprtab’
    tabSearch: no visible binding for global variable ‘Affymoe4302Exprtab’
    tabSearch: no visible binding for global variable ‘Affyhgu133A2Exprtab’
    tabSearch: no visible binding for global variable
      ‘Affyhgu133Plus2Exprtab’
    Undefined global functions or variables:
      Affyhgu133A2Exprtab Affyhgu133Plus2Exprtab Affyhgu133aExprtab
      Affymoe4302Exprtab P.value SampleType Var1 Var2 allreffile
      colorRampPalette data dev.off fisher.test geneid hist par pdf qnorm
      quantile sd str t.stat t.test title value variable write.csv
      write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf")
      importFrom("graphics", "hist", "par", "title")
      importFrom("stats", "fisher.test", "qnorm", "quantile", "sd", "t.test")
      importFrom("utils", "data", "str", "write.csv", "write.table")
    to your NAMESPACE file.
    ```

# GSVA

<details>

* Version: 1.30.0
* Source code: ???
* URL: https://github.com/rcastelo/GSVA
* BugReports: https://github.com/rcastelo/GSVA/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 52

Run `revdep_details(,"GSVA")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
       ‘GSVA.Rnw’ using ‘latin1’ ... failed
     WARNING
    Errors in running code in vignettes:
    when running code in ‘GSVA.Rnw’
      ...
    The following object is masked from ‘package:XML’:
    
        addNode
    
    
    > library(GSVAdata)
    
      When sourcing ‘GSVA.R’:
    Error: there is no package called ‘GSVAdata’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘GSVAdata’
    ```

# highcharter

<details>

* Version: 0.7.0
* Source code: https://github.com/cran/highcharter
* URL: http://jkunst.com/highcharter
* BugReports: https://github.com/jbkunst/highcharter/issues
* Date/Publication: 2019-01-15 16:50:03 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"highcharter")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        doc           3.7Mb
        htmlwidgets   4.0Mb
    ```

# ideal

<details>

* Version: 1.6.1
* Source code: ???
* URL: https://github.com/federicomarini/ideal, https://federicomarini.github.io/ideal/
* BugReports: https://github.com/federicomarini/ideal/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 194

Run `revdep_details(,"ideal")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ideal: no visible binding for '<<-' assignment to ‘ideal_env’
    ideal : <anonymous>: no visible binding for global variable ‘airway’
    ideal : <anonymous>: no visible binding for global variable ‘pvalue’
    ideal : <anonymous>: no visible binding for global variable ‘baseMean’
    ideal : <anonymous>: no visible binding for global variable ‘ideal_env’
    Undefined global functions or variables:
      airway baseMean ideal_env pvalue
    ```

# idefix

<details>

* Version: 0.3.3
* Source code: https://github.com/cran/idefix
* URL: https://github.com/traets/idefix
* Date/Publication: 2018-11-09 14:20:08 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"idefix")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘gtools’ ‘maxLik’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘bayesm’, ‘ChoiceModelR’, ‘RSGHB’, ‘Rchoice’
    ```

# idem

<details>

* Version: 4.0
* Source code: https://github.com/cran/idem
* Date/Publication: 2018-11-09 14:30:22 UTC
* Number of recursive dependencies: 86

Run `revdep_details(,"idem")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# IMWatson

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/IMWatson
* Date/Publication: 2019-03-29 16:40:03 UTC
* Number of recursive dependencies: 50

Run `revdep_details(,"IMWatson")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘V8’ ‘assertive’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# INDperform

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/INDperform
* URL: https://github.com/saskiaotto/INDperform
* BugReports: https://github.com/SaskiaAOtto/INDperform/issues
* Date/Publication: 2019-02-10 03:53:24 UTC
* Number of recursive dependencies: 70

Run `revdep_details(,"INDperform")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   4.0Mb
        help   1.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lazyeval’
      All declared Imports should be used.
    ```

# inlabru

<details>

* Version: 2.1.9
* Source code: https://github.com/cran/inlabru
* URL: http://www.inlabru.org,
* BugReports: https://github.com/fbachl/inlabru/issues
* Date/Publication: 2018-07-24 09:20:03 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"inlabru")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘INLA’
    ```

# INSPEcT

<details>

* Version: 1.12.2
* Source code: ???
* Date/Publication: 2019-04-04
* Number of recursive dependencies: 127

Run `revdep_details(,"INSPEcT")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      1.1Mb
        data   1.6Mb
        doc    1.3Mb
    ```

# interactiveDisplay

<details>

* Version: 1.20.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 151

Run `revdep_details(,"interactiveDisplay")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘.interactiveDisplayBase’
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘rstudio’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for global variable ‘Var1’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘Var2’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘value’
    Undefined global functions or variables:
      AnnotationTrack GO.db GRanges GRangesList GenomeAxisTrack IRanges
      IdeogramTrack MRcounts Var1 Var2 as.dendrogram assays cim
      colorRampPalette coord_flip cutree dendrapply dev.off dist
      elementMetadata experimentData exprs fData hclust installed.packages
      is.leaf layout_circle legend libSize mcols pData plot plotFeature
      plotOrd plotTracks png rainbow ranges rowRanges seqlengths
      seqlengths<- seqlevels<- seqnames ucscGenomes value
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png",
                 "rainbow")
      importFrom("graphics", "legend", "plot")
      importFrom("stats", "as.dendrogram", "cutree", "dendrapply", "dist",
                 "hclust", "is.leaf")
      importFrom("utils", "installed.packages")
    to your NAMESPACE file.
    ```

# interactiveDisplayBase

<details>

* Version: 1.20.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 26

Run `revdep_details(,"interactiveDisplayBase")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    'loadNamespace' or 'requireNamespace' call not declared from: ‘BiocManager’
    ```

*   checking R code for possible problems ... NOTE
    ```
    .usePackage: no visible global function definition for
      ‘install.packages’
    .usePackage: no visible global function definition for
      ‘installed.packages’
    Undefined global functions or variables:
      install.packages installed.packages
    Consider adding
      importFrom("utils", "install.packages", "installed.packages")
    to your NAMESPACE file.
    ```

# interAdapt

<details>

* Version: 0.1
* Source code: https://github.com/cran/interAdapt
* Date/Publication: 2014-08-03 08:03:00
* Number of recursive dependencies: 39

Run `revdep_details(,"interAdapt")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    compute_design_performance : expected_duration_plot: no visible global
      function definition for ‘legend’
    compute_design_performance : boundary_adapt_plot: no visible global
      function definition for ‘matplot’
    compute_design_performance : boundary_adapt_plot: no visible global
      function definition for ‘legend’
    compute_design_performance : boundary_standard_H0C_plot: no visible
      global function definition for ‘matplot’
    compute_design_performance : boundary_standard_H0C_plot: no visible
      global function definition for ‘legend’
    compute_design_performance : boundary_standard_H01_plot: no visible
      global function definition for ‘matplot’
    compute_design_performance : boundary_standard_H01_plot: no visible
      global function definition for ‘legend’
    Undefined global functions or variables:
      legend lines matplot plot read.csv rnorm
    Consider adding
      importFrom("graphics", "legend", "lines", "matplot", "plot")
      importFrom("stats", "rnorm")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# ipumsr

<details>

* Version: 0.4.0
* Source code: https://github.com/cran/ipumsr
* URL: https://www.ipums.org, https://github.com/mnpopcenter/ipumsr
* BugReports: https://github.com/mnpopcenter/ipumsr/issues
* Date/Publication: 2019-03-08 22:30:03 UTC
* Number of recursive dependencies: 96

Run `revdep_details(,"ipumsr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# iSEE

<details>

* Version: 1.2.4
* Source code: ???
* URL: https://github.com/csoneson/iSEE
* BugReports: https://github.com/csoneson/iSEE/issues
* Date/Publication: 2019-04-10
* Number of recursive dependencies: 123

Run `revdep_details(,"iSEE")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 17.9Mb
      sub-directories of 1Mb or more:
        doc  17.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# ivygapSE

<details>

* Version: 1.4.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 113

Run `revdep_details(,"ivygapSE")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        data  13.9Mb
        doc    2.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    exprByType: no visible binding for global variable ‘ivySE’
    ivyGlimpse: no visible binding for global variable ‘ivySE’
    ivyGlimpse : server: no visible binding for global variable ‘ivySE’
    Undefined global functions or variables:
      ivySE
    ```

# jpmesh

<details>

* Version: 1.1.2
* Source code: https://github.com/cran/jpmesh
* URL: https://github.com/uribo/jpmesh#readme
* BugReports: https://github.com/uribo/jpmesh/issues
* Date/Publication: 2019-03-20 23:20:07 UTC
* Number of recursive dependencies: 71

Run `revdep_details(,"jpmesh")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 176 marked UTF-8 strings
    ```

# jpndistrict

<details>

* Version: 0.3.2
* Source code: https://github.com/cran/jpndistrict
* URL: https://github.com/uribo/jpndistrict#readme
* BugReports: https://github.com/uribo/jpndistrict/issues
* Date/Publication: 2018-06-14 13:59:57 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"jpndistrict")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 502 marked UTF-8 strings
    ```

# kokudosuuchi

<details>

* Version: 0.4.2
* Source code: https://github.com/cran/kokudosuuchi
* URL: https://github.com/yutannihilation/kokudosuuchi
* BugReports: https://github.com/yutannihilation/kokudosuuchi/issues
* Date/Publication: 2018-01-26 04:21:18 UTC
* Number of recursive dependencies: 49

Run `revdep_details(,"kokudosuuchi")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 52458 marked UTF-8 strings
    ```

# koRpus

<details>

* Version: 0.11-5
* Source code: https://github.com/cran/koRpus
* URL: https://reaktanz.de/?c=hacking&s=koRpus
* BugReports: https://github.com/unDocUMeantIt/koRpus/issues
* Date/Publication: 2018-10-28 07:50:02 UTC
* Number of recursive dependencies: 47

Run `revdep_details(,"koRpus")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘koRpus.lang.de’ ‘koRpus.lang.es’ ‘koRpus.lang.fr’ ‘koRpus.lang.it’
      ‘koRpus.lang.ru’
    
    Package which this enhances but not available for checking: ‘rkward’
    ```

# learnstats

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/learnstats
* Date/Publication: 2015-06-25 07:20:19
* Number of recursive dependencies: 110

Run `revdep_details(,"learnstats")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    qqsim: no visible global function definition for ‘mtext’
    qqsim: no visible global function definition for ‘jpeg’
    qqsim: no visible global function definition for ‘dev.off’
    timeseressim: no visible global function definition for ‘rnorm’
    timeseressim: no visible global function definition for ‘par’
    timeseressim: no visible global function definition for ‘plot’
    timeseressim: no visible global function definition for ‘mtext’
    timeseressim: no visible global function definition for ‘jpeg’
    timeseressim: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      axis dchisq dev.off dnorm dt hist jpeg legend mtext par pchisq pf
      plot pnorm polygon pt qchisq qf qqline qqnorm qt rchisq rnorm rt
      runif segments title
    Consider adding
      importFrom("grDevices", "dev.off", "jpeg")
      importFrom("graphics", "axis", "hist", "legend", "mtext", "par",
                 "plot", "polygon", "segments", "title")
      importFrom("stats", "dchisq", "dnorm", "dt", "pchisq", "pf", "pnorm",
                 "pt", "qchisq", "qf", "qqline", "qqnorm", "qt", "rchisq",
                 "rnorm", "rt", "runif")
    to your NAMESPACE file.
    ```

# levi

<details>

* Version: 1.0.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 67

Run `revdep_details(,"levi")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    LEVIui: no visible global function definition for ‘is’
    levi_function: no visible binding for global variable ‘x’
    Undefined global functions or variables:
      is x
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# likert

<details>

* Version: 1.3.5
* Source code: https://github.com/cran/likert
* URL: http://jason.bryer.org/likert, http://github.com/jbryer/likert
* BugReports: https://github.com/jbryer/likert/issues
* Date/Publication: 2016-12-31 01:37:00
* Number of recursive dependencies: 78

Run `revdep_details(,"likert")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

# listviewer

<details>

* Version: 2.1.0
* Source code: https://github.com/cran/listviewer
* URL: https://github.com/timelyportfolio/listviewer
* BugReports: https://github.com/timelyportfolio/listviewer/issues
* Date/Publication: 2018-10-07 04:30:02 UTC
* Number of recursive dependencies: 20

Run `revdep_details(,"listviewer")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘reactR’
    ```

# live

<details>

* Version: 1.5.10
* Source code: https://github.com/cran/live
* URL: https://github.com/MI2DataLab/live
* BugReports: https://github.com/MI2DataLab/live/issues
* Date/Publication: 2019-02-28 13:00:03 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"live")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘e1071’
      All declared Imports should be used.
    ```

# lmviz

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/lmviz
* Date/Publication: 2018-11-18 18:30:10 UTC
* Number of recursive dependencies: 23

Run `revdep_details(,"lmviz")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyjs’
      All declared Imports should be used.
    ```

# mapdeck

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/mapdeck
* URL: https://symbolixau.github.io/mapdeck/articles/mapdeck.html
* BugReports: https://github.com/SymbolixAU/mapdeck/issues
* Date/Publication: 2019-01-22 09:20:03 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"mapdeck")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc           1.2Mb
        htmlwidgets   1.4Mb
        libs          3.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# mapedit

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/mapedit
* URL: https://github.com/r-spatial/mapedit
* BugReports: https://github.com/r-spatial/mapedit/issues
* Date/Publication: 2019-03-16 22:50:08 UTC
* Number of recursive dependencies: 77

Run `revdep_details(,"mapedit")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘geojsonio’
    ```

# mbgraphic

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/mbgraphic
* Date/Publication: 2017-05-13 10:15:21 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"mbgraphic")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘extracat’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# memapp

<details>

* Version: 2.12
* Source code: https://github.com/cran/memapp
* URL: https://github.com/lozalojo/memapp
* BugReports: https://github.com/lozalojo/memapp/issues
* Date/Publication: 2019-01-29 12:50:03 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"memapp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘RODBC’ ‘dplyr’ ‘foreign’ ‘formattable’ ‘ggplot2’
      ‘haven’ ‘mem’ ‘openxlsx’ ‘plotly’ ‘readxl’ ‘shinyBS’ ‘shinydashboard’
      ‘shinydashboardPlus’ ‘shinyjs’ ‘shinythemes’ ‘stringi’ ‘stringr’
      ‘tidyr’
      All declared Imports should be used.
    ```

# memery

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/memery
* URL: https://github.com/leonawicz/memery
* BugReports: https://github.com/leonawicz/memery/issues
* Date/Publication: 2018-05-17 22:45:52 UTC
* Number of recursive dependencies: 81

Run `revdep_details(,"memery")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘shinyBS’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# meta4diag

<details>

* Version: 2.0.8
* Source code: https://github.com/cran/meta4diag
* Date/Publication: 2018-07-23 12:00:02 UTC
* Number of recursive dependencies: 22

Run `revdep_details(,"meta4diag")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# MetamapsDB

<details>

* Version: 0.0.2
* Source code: https://github.com/cran/MetamapsDB
* Date/Publication: 2017-12-06 09:51:00 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"MetamapsDB")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘shiny’
      All declared Imports should be used.
    ```

# MetCirc

<details>

* Version: 1.12.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 36

Run `revdep_details(,"MetCirc")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      ========================================
      
      > library(scales)
      > library(shiny)
      > 
      > data("sd01_outputXCMS", package = "MetCirc")
      > data("sd02_deconvoluted", package = "MetCirc")
      > data("binnedMSP", package = "MetCirc")
      > data("similarityMat", package = "MetCirc")
      > 
      > BiocGenerics:::testPackage("MetCirc")
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘circlize:::get.sector.data’
      See the note in ?`:::` about the use of this operator.
    ```

# MethylAid

<details>

* Version: 1.16.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 146

Run `revdep_details(,"MethylAid")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MethylAid-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: summarize
    > ### Title: summarization of Illumina Human DNA Methylation array data
    > ### Aliases: summarize
    > 
    > ### ** Examples
    > 
    > library(minfiData)
    Error in library(minfiData) : there is no package called ‘minfiData’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      ERROR in test_summarize: Error in library(minfiData) : there is no package called 'minfiData'
      ERROR in test_visualize: Error in library(minfiData) : there is no package called 'minfiData'
      
      Test files with failing tests
      
         test_summarize.R 
           test_summarize 
      
         test_visualize.R 
           test_visualize 
      
      
      Error in BiocGenerics:::testPackage("MethylAid") : 
        unit tests failed for package MethylAid
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘MethylAidData’ ‘minfiData’ ‘minfiDataEPIC’
    ```

# miRcomp

<details>

* Version: 1.12.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 36

Run `revdep_details(,"miRcomp")` for more info

</details>

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                      old_size new_size compress
      lifetech.rda       350Kb    276Kb       xz
      qpcRb4.rda         290Kb    244Kb       xz
      qpcRb5.rda         288Kb    242Kb       xz
      qpcRdefault.rda    290Kb    242Kb       xz
      qpcRl5.rda         287Kb    239Kb       xz
      qpcRlinexp.rda     235Kb    199Kb       xz
    ```

# MLInterfaces

<details>

* Version: 1.62.1
* Source code: ???
* Date/Publication: 2019-04-03
* Number of recursive dependencies: 144

Run `revdep_details(,"MLInterfaces")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > c3 = MLearn(~CL+CW+RW, crabs, pamI(dist), k=5)
    > c3
    clusteringOutput: partition table
    
     1  2  3  4  5 
    30 48 43 52 27 
    The call that created this object was:
    MLearn(formula = ~CL + CW + RW, data = crabs, .method = pamI(dist), 
        k = 5)
    > plot(c3, data=crabs[,c("CL", "CW", "RW")])
    > 
    > 
    > #  new interfaces to PLS thanks to Laurent Gatto
    > 
    > set.seed(1234)
    > kp = sample(1:200, size=120)
    > 
    > plsda.1 = MLearn(sp~CW+RW, data=crabs, plsdaI, kp, probMethod="Bayes")
    Error: package klaR is required
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
          Vignettes contain introductory material; view with
          'browseVignettes()'. To cite Bioconductor, see
          'citation("Biobase")', and for packages 'citation("pkgname")'.
      
      
      Attaching package: 'S4Vectors'
      
      The following object is masked from 'package:base':
      
          expand.grid
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘MASS:::predict.lda’ ‘ada:::predict.ada’ ‘e1071:::predict.svm’
      ‘rpart:::predict.rpart’ ‘stats:::plot.hclust’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    hclustWidget : <anonymous>: no visible global function definition for
      ‘:=’
    hclustWidget : <anonymous>: no visible binding for global variable
      ‘key’
    Undefined global functions or variables:
      := key
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: plspinHcube.Rd:17-19: Dropping empty section \details
    ```

# MMDiff2

<details>

* Version: 1.10.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 87

Run `revdep_details(,"MMDiff2")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BSgenome.Mmusculus.UCSC.mm9’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    DBAmmd: no visible global function definition for ‘new’
    getReads: no visible global function definition for ‘IRangesList’
    Undefined global functions or variables:
      IRangesList new
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# MNLR

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MNLR
* Date/Publication: 2019-03-23 10:06:11 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"MNLR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘e1071’ ‘nnet’ ‘shiny’ ‘stats’
      All declared Imports should be used.
    ```

# MODIStsp

<details>

* Version: 1.3.8
* Source code: https://github.com/cran/MODIStsp
* URL: https://github.com/ropensci/MODIStsp
* BugReports: https://github.com/ropensci/MODIStsp/issues
* Date/Publication: 2019-03-08 12:02:44 UTC
* Number of recursive dependencies: 115

Run `revdep_details(,"MODIStsp")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘gWidgetsRGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MSGFgui

<details>

* Version: 1.16.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 55

Run `revdep_details(,"MSGFgui")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘xlsx’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    renderMzID: no visible global function definition for ‘modifications’
    renderMzID: no visible global function definition for ‘database’
    server: no visible global function definition for ‘scans’
    server : <anonymous>: no visible global function definition for
      ‘createWorkbook’
    server : <anonymous>: no visible global function definition for
      ‘createSheet’
    server : <anonymous>: no visible global function definition for
      ‘addDataFrame’
    server : <anonymous>: no visible global function definition for
      ‘saveWorkbook’
    server : <anonymous>: no visible global function definition for
      ‘write.table’
    Undefined global functions or variables:
      addDataFrame createSheet createWorkbook database density dnorm loess
      modifications optimize peptides predict read.csv saveWorkbook scans
      write.table
    Consider adding
      importFrom("stats", "density", "dnorm", "loess", "optimize", "predict")
      importFrom("utils", "read.csv", "write.table")
    to your NAMESPACE file.
    ```

# MSnbase

<details>

* Version: 2.8.3
* Source code: ???
* URL: https://github.com/lgatto/MSnbase
* BugReports: https://github.com/lgatto/MSnbase/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 215

Run `revdep_details(,"MSnbase")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MSnbase-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: FeatComp-class
    > ### Title: Class '"FeatComp"'
    > ### Aliases: FeatComp-class compfnames-methods
    > ###   compfnames,MSnSet,MSnSet-method compfnames,list,missing-method
    > ###   compfnames show,FeatComp-method names,FeatComp-method
    > ###   common,FeatComp-method common,methods common unique1,FeatComp-method
    > ###   unique1,methods unique1 unique2,FeatComp-method unique2,methods
    > ###   unique2
    > ### Keywords: classes
    > 
    > ### ** Examples
    > 
    > library("pRolocdata")
    Error in library("pRolocdata") : there is no package called ‘pRolocdata’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3136 SKIPPED: 9 FAILED: 11
      1. Error: MSnSet coersion (@test_MSnSet.R#25) 
      2. Error: commonFeatureNames works with lists or MSnSetLists (@test_MSnSet.R#358) 
      3. Error: keeping common features (@test_MSnSet.R#377) 
      4. Error: nFeatures are added correctly (@test_MSnSet.R#458) 
      5. Error: (unknown) (@test_average.R#1) 
      6. Error: Feature variable selection (@test_fdata-selection.R#4) 
      7. Error: (unknown) (@test_foi.R#1) 
      8. Error: readMSnSet2: rownames and fnames (@test_io.R#84) 
      9. Error: (unknown) (@test_nadata.R#2) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pRolocdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.4Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.2Mb
        doc    7.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
      Johannes Rainer <Johannes.Rainer@eurac.edu> [aut, cre]
      Sebastian Gibb <mail@sebastiangibb.de> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘Biobase:::.showAnnotatedDataFrame’ ‘MALDIquant:::.estimateNoise’
      ‘MALDIquant:::.localMaxima’ ‘MALDIquant:::.movingAverage’
      ‘MALDIquant:::.savitzkyGolay’
      ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
      ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
      ‘mzR:::.hasChromatograms’ ‘mzR:::.hasSpectra’
      See the note in ?`:::` about the use of this operator.
    ```

# MuChPoint

<details>

* Version: 0.6.1
* Source code: https://github.com/cran/MuChPoint
* Date/Publication: 2018-06-24 13:09:46 UTC
* Number of recursive dependencies: 20

Run `revdep_details(,"MuChPoint")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘capushe’ ‘shiny’
      All declared Imports should be used.
    ```

# nbc4va

<details>

* Version: 1.1
* Source code: https://github.com/cran/nbc4va
* Date/Publication: 2018-08-05 16:40:03 UTC
* Number of recursive dependencies: 34

Run `revdep_details(,"nbc4va")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘openVA’
    ```

# NDP

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/NDP
* Date/Publication: 2019-03-30 15:00:02 UTC
* Number of recursive dependencies: 28

Run `revdep_details(,"NDP")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# neo4r

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/neo4r
* URL: https://github.com/neo4j-rstats/neo4r
* BugReports: https://github.com/neo4j-rstats/neo4r/issues
* Date/Publication: 2019-02-15 22:20:03 UTC
* Number of recursive dependencies: 41

Run `revdep_details(,"neo4r")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘igraph’ ‘rlang’ ‘tidyselect’
      All declared Imports should be used.
    ```

# objectremover

<details>

* Version: 0.6.4
* Source code: https://github.com/cran/objectremover
* URL: https://github.com/alan-y/objectremover
* BugReports: https://github.com/alan-y/objectremover/issues
* Date/Publication: 2019-03-06 16:50:04 UTC
* Number of recursive dependencies: 18

Run `revdep_details(,"objectremover")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstudioapi’
      All declared Imports should be used.
    ```

# omicplotR

<details>

* Version: 1.2.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 70

Run `revdep_details(,"omicplotR")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    omicplotr.colvec: no visible global function definition for ‘col2rgb’
    Undefined global functions or variables:
      col2rgb
    Consider adding
      importFrom("grDevices", "col2rgb")
    to your NAMESPACE file.
    ```

# oneSENSE

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 93

Run `revdep_details(,"oneSENSE")` for more info

</details>

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'export' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

# ontoProc

<details>

* Version: 1.4.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 53

Run `revdep_details(,"ontoProc")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 28.0Mb
      sub-directories of 1Mb or more:
        ontoRda  26.9Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 162 marked UTF-8 strings
    ```

# openPrimeRui

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2018-11-10
* Number of recursive dependencies: 117

Run `revdep_details(,"openPrimeRui")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘openPrimeR’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# oppr

<details>

* Version: 0.0.3
* Source code: https://github.com/cran/oppr
* URL: https://prioritizr.github.io/oppr, https://github.com/prioritizr/oppr
* BugReports: https://github.com/prioritizr/oppr/issues
* Date/Publication: 2019-04-10 21:42:59 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"oppr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘gurobi’ ‘Rsymphony’
    ```

# pairsD3

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/pairsD3
* URL: http://github.com/garthtarr/pairsD3/
* Date/Publication: 2015-04-28 06:58:01
* Number of recursive dependencies: 26

Run `revdep_details(,"pairsD3")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    pairsD3: no visible global function definition for ‘gray.colors’
    Undefined global functions or variables:
      gray.colors
    Consider adding
      importFrom("grDevices", "gray.colors")
    to your NAMESPACE file.
    ```

# PanVizGenerator

<details>

* Version: 1.10.0
* Source code: ???
* URL: https://github.com/thomasp85/PanVizGenerator
* BugReports: https://github.com/thomasp85/PanVizGenerator/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 85

Run `revdep_details(,"PanVizGenerator")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# PathoStat

<details>

* Version: 1.8.4
* Source code: ???
* URL: https://github.com/mani2012/PathoStat
* BugReports: https://github.com/mani2012/PathoStat/issues
* Date/Publication: 2018-12-02
* Number of recursive dependencies: 177

Run `revdep_details(,"PathoStat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ComplexHeatmap’ ‘RColorBrewer’
      All declared Imports should be used.
    ```

# pcadapt

<details>

* Version: 4.1.0
* Source code: https://github.com/cran/pcadapt
* Date/Publication: 2019-02-27 13:20:03 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"pcadapt")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   2.7Mb
    ```

# pcaExplorer

<details>

* Version: 2.8.1
* Source code: ???
* URL: https://github.com/federicomarini/pcaExplorer, https://federicomarini.github.io/pcaExplorer/
* BugReports: https://github.com/federicomarini/pcaExplorer/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 170

Run `revdep_details(,"pcaExplorer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        doc   6.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    pcaExplorer: no visible binding for '<<-' assignment to
      ‘pcaexplorer_env’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘airway’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘pcaexplorer_env’
    Undefined global functions or variables:
      airway pcaexplorer_env
    ```

# pepStat

<details>

* Version: 1.16.0
* Source code: ???
* URL: https://github.com/RGLab/pepStat
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 125

Run `revdep_details(,"pepStat")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘pepStat’ for: ‘end’, ‘start’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,peptideSet-ExpressionSet: no visible global function definition
      for ‘annotation’
    end,peptideSet: no visible global function definition for ‘end’
    position,peptideSet: no visible global function definition for ‘start’
    position,peptideSet: no visible global function definition for ‘end’
    start,peptideSet: no visible global function definition for ‘start’
    write.pSet,peptideSet: no visible global function definition for
      ‘start’
    write.pSet,peptideSet: no visible global function definition for ‘end’
    write.pSet,peptideSet: no visible global function definition for
      ‘write.csv’
    Undefined global functions or variables:
      annotation dev.flush dev.hold dev.interactive devAskNewPage end
      lm.fit lm.wfit mcols mcols<- median read.csv sd start write.csv
    Consider adding
      importFrom("grDevices", "dev.flush", "dev.hold", "dev.interactive",
                 "devAskNewPage")
      importFrom("stats", "end", "lm.fit", "lm.wfit", "median", "sd",
                 "start")
      importFrom("utils", "read.csv", "write.csv")
    to your NAMESPACE file.
    ```

# periscope

<details>

* Version: 0.4.1
* Source code: https://github.com/cran/periscope
* URL: https://github.com/cb4ds/periscope.git, http://canvasxpress.org
* BugReports: https://github.com/cb4ds/periscope/issues
* Date/Publication: 2019-03-06 17:30:07 UTC
* Number of recursive dependencies: 68

Run `revdep_details(,"periscope")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘httr’
      All declared Imports should be used.
    ```

# phenocamr

<details>

* Version: 1.1.3
* Source code: https://github.com/cran/phenocamr
* Date/Publication: 2019-02-26 20:50:07 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"phenocamr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘jsonlite’ ‘memoise’
      All declared Imports should be used.
    ```

# pitchRx

<details>

* Version: 1.8.2
* Source code: https://github.com/cran/pitchRx
* URL: http://cpsievert.github.com/pitchRx
* BugReports: http://github.com/cpsievert/pitchRx/issues
* Date/Publication: 2015-12-09 12:57:25
* Number of recursive dependencies: 82

Run `revdep_details(,"pitchRx")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ggsubplot’
    ```

# pivottabler

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/pivottabler
* URL: https://github.com/cbailiss/pivottabler
* BugReports: https://github.com/cbailiss/pivottabler/issues
* Date/Publication: 2019-03-20 23:30:06 UTC
* Number of recursive dependencies: 65

Run `revdep_details(,"pivottabler")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.0Mb
        doc    3.0Mb
    ```

# plethem

<details>

* Version: 0.1.7
* Source code: https://github.com/cran/plethem
* Date/Publication: 2019-01-02 15:50:03 UTC
* Number of recursive dependencies: 131

Run `revdep_details(,"plethem")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘V8’ ‘devtools’ ‘formatR’ ‘gdata’ ‘rhandsontable’ ‘shinythemes’
      ‘sqldf’
      All declared Imports should be used.
    ```

# plotGrouper

<details>

* Version: 1.0.1
* Source code: ???
* URL: https://jdgagnon.github.io/plotGrouper/
* BugReports: https://github.com/jdgagnon/plotGrouper/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 98

Run `revdep_details(,"plotGrouper")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    gplot: no visible binding for global variable ‘max_value’
    gplot: no visible binding for global variable ‘max_error’
    Undefined global functions or variables:
      max_error max_value
    ```

# plotly

<details>

* Version: 4.9.0
* Source code: https://github.com/cran/plotly
* URL: https://plotly-r.com, https://github.com/ropensci/plotly#readme, https://plot.ly/r
* BugReports: https://github.com/ropensci/plotly/issues
* Date/Publication: 2019-04-10 19:33:05 UTC
* Number of recursive dependencies: 150

Run `revdep_details(,"plotly")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        htmlwidgets   3.4Mb
    ```

# pmd

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/pmd
* URL: https://yufree.github.io/pmd
* BugReports: https://github.com/yufree/pmd/issues
* Date/Publication: 2018-12-02 17:20:03 UTC
* Number of recursive dependencies: 123

Run `revdep_details(,"pmd")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# pogos

<details>

* Version: 1.2.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 79

Run `revdep_details(,"pogos")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    compoundsByCell: no visible binding for global variable ‘cell_lines_v1’
    compoundsByCell: no visible binding for global variable ‘compounds_v1’
    compoundsByCell: no visible binding for global variable ‘datasets_v1’
    lkc: no visible binding for global variable ‘cell_lines_v1’
    lkc: no visible binding for global variable ‘compounds_v1’
    lkc: no visible binding for global variable ‘datasets_v1’
    plot,DRProfSet-missing: no visible binding for global variable ‘dose’
    plot,DRProfSet-missing: no visible binding for global variable
      ‘response’
    plot,DRProfSet-missing: no visible binding for global variable ‘drug’
    plot,DRTraceSet-missing: no visible binding for global variable ‘dose’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘response’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘cell_line’
    Undefined global functions or variables:
      cell_line cell_lines_v1 compounds_v1 datasets_v1 dose drug response
    ```

# PopED

<details>

* Version: 0.4.0
* Source code: https://github.com/cran/PopED
* URL: http://poped.sourceforge.net
* BugReports: https://github.com/andrewhooker/PopED/issues
* Date/Publication: 2018-09-10 09:40:03 UTC
* Number of recursive dependencies: 92

Run `revdep_details(,"PopED")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# poppr

<details>

* Version: 2.8.2
* Source code: https://github.com/cran/poppr
* URL: https://github.com/grunwaldlab/poppr, https://grunwaldlab.github.io/poppr, https://grunwaldlab.github.io/Population_Genetics_in_R/
* BugReports: https://github.com/grunwaldlab/poppr/issues
* Date/Publication: 2019-03-11 10:30:03 UTC
* Number of recursive dependencies: 93

Run `revdep_details(,"poppr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        R     2.1Mb
        doc   2.4Mb
    ```

# powdR

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/powdR
* URL: http://github.com/benmbutler/powdR
* BugReports: http://github.com/benmbutler/powdR/issues
* Date/Publication: 2018-11-15 17:50:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"powdR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyWidgets’
      All declared Imports should be used.
    ```

# predictoR

<details>

* Version: 1.0.4
* Source code: https://github.com/cran/predictoR
* URL: http://www.promidat.com
* Date/Publication: 2019-03-03 11:20:03 UTC
* Number of recursive dependencies: 126

Run `revdep_details(,"predictoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ROCR’ ‘ada’ ‘colourpicker’ ‘corrplot’ ‘e1071’ ‘flexdashboard’
      ‘kknn’ ‘neuralnet’ ‘randomForest’ ‘rattle’ ‘rpart’ ‘scatterplot3d’
      ‘shinyAce’ ‘shinyWidgets’ ‘shinydashboardPlus’ ‘shinyjs’ ‘tidyverse’
      ‘xgboost’ ‘zip’
      All declared Imports should be used.
    ```

# primirTSS

<details>

* Version: 1.0.1
* Source code: ???
* URL: https://github.com/ipumin/primirTSS
* BugReports: http://github.com/ipumin/primirTSS/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 149

Run `revdep_details(,"primirTSS")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      ‘BSgenome.Hsapiens.UCSC.hg38’ ‘phastCons100way.UCSC.hg38’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# prioritizr

<details>

* Version: 4.0.4
* Source code: https://github.com/cran/prioritizr
* URL: https://prioritizr.net, https://github.com/prioritizr/prioritizr
* BugReports: https://github.com/prioritizr/prioritizr/issues
* Date/Publication: 2019-04-12 12:32:42 UTC
* Number of recursive dependencies: 100

Run `revdep_details(,"prioritizr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘gurobi’ ‘Rsymphony’
    ```

# processanimateR

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/processanimateR
* URL: https://github.com/fmannhardt/processanimateR/
* BugReports: https://github.com/fmannhardt/processanimateR/issues
* Date/Publication: 2018-11-27 13:30:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"processanimateR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc           6.5Mb
        help          2.1Mb
        htmlwidgets   2.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘zoo’
      All declared Imports should be used.
    ```

# processR

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/processR
* URL: https://github.com/cardiomoon/processR
* BugReports: https://github.com/cardiomoon/processR/issues
* Date/Publication: 2019-03-07 17:40:03 UTC
* Number of recursive dependencies: 117

Run `revdep_details(,"processR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘TH.data’ ‘jtools’ ‘modelr’ ‘prediction’ ‘rlang’ ‘tidyr’
      All declared Imports should be used.
    ```

# pRolocGUI

<details>

* Version: 1.16.0
* Source code: ???
* URL: http://ComputationalProteomicsUnit.github.io/pRolocGUI/
* BugReports: https://github.com/ComputationalProteomicsUnit/pRolocGUI/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 174

Run `revdep_details(,"pRolocGUI")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘pRolocGUI-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: pRolocVis
    > ### Title: Interactive visualisation of spatial proteomics data
    > ### Aliases: pRolocVis pRolocVis_aggregate pRolocVis_classify
    > ###   pRolocVis_compare pRolocVis_pca
    > 
    > ### ** Examples
    > 
    > library("pRoloc")
    > library("pRolocdata")
    Error in library("pRolocdata") : there is no package called ‘pRolocdata’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pRolocdata’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Lisa Breckels <lms79@cam.ac.uk> [aut, cre]
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘pRoloc:::remap’
      See the note in ?`:::` about the use of this operator.
    ```

# psichomics

<details>

* Version: 1.8.2
* Source code: ???
* URL: https://github.com/nuno-agostinho/psichomics
* BugReports: https://github.com/nuno-agostinho/psichomics/issues
* Date/Publication: 2019-03-27
* Number of recursive dependencies: 220

Run `revdep_details(,"psichomics")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        R     1.0Mb
        doc   5.1Mb
    ```

# QCA

<details>

* Version: 3.4
* Source code: https://github.com/cran/QCA
* Date/Publication: 2019-01-18 09:10:03 UTC
* Number of recursive dependencies: 18

Run `revdep_details(,"QCA")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘cna’
    ```

# qlcData

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/qlcData
* Date/Publication: 2018-01-06 00:00:34 UTC
* Number of recursive dependencies: 92

Run `revdep_details(,"qlcData")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1459 marked UTF-8 strings
    ```

# qqplotr

<details>

* Version: 0.0.3
* Source code: https://github.com/cran/qqplotr
* URL: https://github.com/aloy/qqplotr
* BugReports: https://github.com/aloy/qqplotr/issues
* Date/Publication: 2018-07-18 08:00:03 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"qqplotr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘purrr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# questionr

<details>

* Version: 0.7.0
* Source code: https://github.com/cran/questionr
* URL: https://juba.github.io/questionr/
* BugReports: https://github.com/juba/questionr/issues
* Date/Publication: 2018-11-26 13:10:06 UTC
* Number of recursive dependencies: 72

Run `revdep_details(,"questionr")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘Hmisc’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4145 marked UTF-8 strings
    ```

# R.SamBada

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/R.SamBada
* Date/Publication: 2019-04-12 08:52:40 UTC
* Number of recursive dependencies: 122

Run `revdep_details(,"R.SamBada")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘R.SamBada-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: downloadSambada
    > ### Title: Download samBada
    > ### Aliases: downloadSambada
    > 
    > ### ** Examples
    > 
    > # Downloads SamBada to temporary folder (tempdir)
    > downloadSambada(tempdir())
    This funcion will download Sambada in the specified or the current directory. Press any letter to continue or x to exit the process 
    Error in downloadSambada(tempdir()) : 
      Unknown operating system. Please download sambada manually from GitHub
    Execution halted
    ```

# R3CPET

<details>

* Version: 1.14.0
* Source code: ???
* BugReports: https://github.com/sirusb/R3CPET/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 145

Run `revdep_details(,"R3CPET")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data      3.0Mb
        example   1.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'BiocGenerics'
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .formatDAVIDResult: no visible global function definition for
      'formatList'
    .formatDAVIDResult: no visible global function definition for
      'formatGene2Gene'
    .formatDAVIDResult: no visible global function definition for
      'formatAnnotationReport'
    .get.NetworksGenes: no visible global function definition for
      'annotatePeakInBatch'
    EnsemblToHGNC: no visible global function definition for 'useMart'
    EnsemblToHGNC: no visible global function definition for 'useDataset'
    EnsemblToHGNC: no visible global function definition for 'getBM'
    EntrezToHGNC: no visible global function definition for 'useMart'
    EntrezToHGNC: no visible global function definition for 'useDataset'
    EntrezToHGNC: no visible global function definition for 'getBM'
    createServer,ChiapetExperimentData-NetworkCollection-ChromMaintainers:
      no visible global function definition for 'runApp'
    Undefined global functions or variables:
      TxDb.Hsapiens.UCSC.hg19.knownGene annotatePeakInBatch
      formatAnnotationReport formatGene2Gene formatGeneReport
      formatGeneReportFull formatList getBM org.Hs.eg.db org.Hs.egUCSCKG
      runApp select toTable useDataset useMart
    ```

# RagGrid

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/RagGrid
* URL: https://github.com/no-types/RagGrid/
* BugReports: https://github.com/no-types/RagGrid/issues
* Date/Publication: 2018-08-12 09:30:03 UTC
* Number of recursive dependencies: 54

Run `revdep_details(,"RagGrid")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# rAmCharts

<details>

* Version: 2.1.10
* Source code: https://github.com/cran/rAmCharts
* URL: http://datastorm-open.github.io/introduction_ramcharts/
* BugReports: https://github.com/datastorm-open/rAmCharts/issues/
* Date/Publication: 2019-02-12 11:10:03 UTC
* Number of recursive dependencies: 38

Run `revdep_details(,"rAmCharts")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 14.1Mb
      sub-directories of 1Mb or more:
        api           3.1Mb
        htmlwidgets   9.2Mb
    ```

# Rariant

<details>

* Version: 1.18.0
* Source code: ???
* URL: https://github.com/juliangehring/Rariant
* BugReports: https://support.bioconductor.org
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 148

Run `revdep_details(,"Rariant")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: tallyPlot
    > 
    > ### ** Examples
    > 
    >   library(ggbio)
    Loading required package: ggplot2
    Need specific help about ggbio? try mailing 
     the maintainer or visit http://tengfei.github.com/ggbio/
    
    Attaching package: 'ggbio'
    
    The following objects are masked from 'package:ggplot2':
    
        geom_bar, geom_rect, geom_segment, ggsave, stat_bin, stat_identity,
        xlim
    
    >   library(GenomicRanges)
    >   library(BSgenome.Hsapiens.UCSC.hg19)
    Error in library(BSgenome.Hsapiens.UCSC.hg19) : 
      there is no package called 'BSgenome.Hsapiens.UCSC.hg19'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BSgenome.Hsapiens.UCSC.hg19’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        doc       2.3Mb
        extdata   5.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    tallyBamRegion: no visible global function definition for 'PileupParam'
    tallyBamRegion: no visible global function definition for
      'ScanBamParam'
    tallyBamRegion: no visible global function definition for 'pileup'
    Undefined global functions or variables:
      PileupParam ScanBamParam pileup
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘rariant-inspect-ci.png’, ‘rariant-inspect-shift.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

# rbin

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/rbin
* URL: https://github.com/rsquaredacademy/rbin, https://rbin.rsquaredacademy.com
* BugReports: https://github.com/rsquaredacademy/rbin/issues
* Date/Publication: 2019-01-04 11:20:04 UTC
* Number of recursive dependencies: 124

Run `revdep_details(,"rbin")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# rblt

<details>

* Version: 0.2.4.0
* Source code: https://github.com/cran/rblt
* URL: https://github.com/sg4r/rblt
* BugReports: https://github.com/sg4r/rblt/issues
* Date/Publication: 2019-04-09 21:32:57 UTC
* Number of recursive dependencies: 35

Run `revdep_details(,"rblt")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘h5’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# rcellminer

<details>

* Version: 2.4.0
* Source code: ???
* URL: http://discover.nci.nih.gov/cellminer/
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 110

Run `revdep_details(,"rcellminer")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.6Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    6.5Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘rcellminer’ for: ‘initialize’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotDrugSets: no visible binding for global variable ‘sd’
    plotDrugSets: no visible global function definition for ‘pdf’
    plotDrugSets: no visible global function definition for ‘par’
    plotDrugSets: no visible global function definition for ‘axis’
    plotDrugSets: no visible global function definition for ‘segments’
    plotDrugSets: no visible global function definition for ‘dev.off’
    plotStructures: no visible global function definition for ‘par’
    rcdkplot: no visible global function definition for ‘par’
    rcdkplot: no visible global function definition for ‘get.depictor’
    rcdkplot: no visible global function definition for ‘plot’
    rcdkplot: no visible global function definition for ‘rasterImage’
    Undefined global functions or variables:
      IQR axis dev.off element_text get.depictor layout lcm lm na.exclude
      par pdf plot quantile rasterImage residuals sd segments xlim ylim
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "axis", "layout", "lcm", "par", "plot",
                 "rasterImage", "segments")
      importFrom("stats", "IQR", "lm", "na.exclude", "quantile", "residuals",
                 "sd")
    to your NAMESPACE file.
    ```

# rCGH

<details>

* Version: 1.12.0
* Source code: ???
* URL: https://github.com/fredcommo/rCGH
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 113

Run `revdep_details(,"rCGH")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        extdata   1.2Mb
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# RDML

<details>

* Version: 0.9-9
* Source code: https://github.com/cran/RDML
* URL: https://github.com/kablag/RDML
* Date/Publication: 2017-09-25 12:07:17 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"RDML")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.4Mb
    ```

# recmap

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/recmap
* BugReports: https://github.com/cpanse/recmap/issues
* Date/Publication: 2019-03-07 01:00:03 UTC
* Number of recursive dependencies: 45

Run `revdep_details(,"recmap")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1036 marked UTF-8 strings
    ```

# RefNet

<details>

* Version: 1.18.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 63

Run `revdep_details(,"RefNet")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# ReportingTools

<details>

* Version: 2.22.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 150

Run `revdep_details(,"ReportingTools")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      ERROR in /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqResults.R: Error while sourcing  /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqResults.R : Error in .requirePackage(package) : 
        unable to find required package 'DESeq'
      
      Test files with failing tests
      
         test_DESeqDataSet.R 
           /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqDataSet.R 
      
         test_DESeqResults.R 
           /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqResults.R 
      
      
      Error in BiocGenerics:::testPackage("ReportingTools") : 
        unit tests failed for package ReportingTools
      Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      'fData'
    .marrayLM.to.html: no visible global function definition for
      'featureNames'
    .marrayLM.to.html: no visible global function definition for 'fData'
    check.ids: no visible binding for global variable 'org.Hs.eg.db'
    check.ids: no visible global function definition for 'keys'
    custHeaderPanel : <anonymous>: no visible binding for global variable
      'tags'
    custHeaderPanel : <anonymous>: no visible global function definition
      for 'HTML'
    custHeaderPanel: no visible global function definition for 'tagList'
    custHeaderPanel: no visible global function definition for 'tag'
    custHeaderPanel: no visible global function definition for 'div'
    custHeaderPanel: no visible global function definition for 'h1'
    publish,trellis-HTMLReport: no visible binding for global variable
      'htmlRep'
    toReportDF,DESeqDataSet: no visible global function definition for
      'mcols'
    Undefined global functions or variables:
      HTML columns description div exprs fData featureNames h1 htmlRep keys
      keytype mcols org.Hs.eg.db tag tagList tags
    ```

# reprex

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/reprex
* URL: https://reprex.tidyverse.org, https://github.com/tidyverse/reprex#readme
* BugReports: https://github.com/tidyverse/reprex/issues
* Date/Publication: 2018-09-16 04:30:09 UTC
* Number of recursive dependencies: 75

Run `revdep_details(,"reprex")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tools’
      All declared Imports should be used.
    ```

# revtools

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/revtools
* URL: https://revtools.net
* BugReports: https://github.com/mjwestgate/revtools/issues
* Date/Publication: 2018-11-29 05:30:02 UTC
* Number of recursive dependencies: 70

Run `revdep_details(,"revtools")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘SnowballC’ ‘stringdist’
      All declared Imports should be used.
    ```

# RforProteomics

<details>

* Version: 1.20.0
* Source code: ???
* URL: http://lgatto.github.com/RforProteomics/
* Date/Publication: 2018-11-01
* Number of recursive dependencies: 257

Run `revdep_details(,"RforProteomics")` for more info

</details>

## In both

*   checking whether package ‘RforProteomics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘MSnbase::plot’ by ‘graphics::plot’ when loading ‘RforProteomics’
    See ‘/Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/RforProteomics/new/RforProteomics.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘synapterdata’ ‘pRolocdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        doc  10.8Mb
    ```

# rgl

<details>

* Version: 0.100.19
* Source code: https://github.com/cran/rgl
* URL: https://r-forge.r-project.org/projects/rgl/
* BugReports: https://r-forge.r-project.org/projects/rgl/
* Date/Publication: 2019-03-12 17:50:03 UTC
* Number of recursive dependencies: 77

Run `revdep_details(,"rgl")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        doc     3.6Mb
        fonts   1.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘heplots’, ‘tkrgl’
    ```

# rhandsontable

<details>

* Version: 0.3.7
* Source code: https://github.com/cran/rhandsontable
* URL: http://jrowen.github.io/rhandsontable/
* BugReports: https://github.com/jrowen/rhandsontable/issues
* Date/Publication: 2018-11-20 05:50:03 UTC
* Number of recursive dependencies: 31

Run `revdep_details(,"rhandsontable")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.5Mb
      sub-directories of 1Mb or more:
        doc          10.2Mb
        htmlwidgets   1.2Mb
    ```

# RLumShiny

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/RLumShiny
* URL: https://tzerk.github.io/RLumShiny/
* BugReports: https://github.com/tzerk/RLumShiny/issues
* Date/Publication: 2019-01-11 11:30:03 UTC
* Number of recursive dependencies: 87

Run `revdep_details(,"RLumShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RCarb’ ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    ```

# rmarkdown

<details>

* Version: 1.12
* Source code: https://github.com/cran/rmarkdown
* URL: https://rmarkdown.rstudio.com
* BugReports: https://github.com/rstudio/rmarkdown/issues
* Date/Publication: 2019-03-14 08:50:03 UTC
* Number of recursive dependencies: 47

Run `revdep_details(,"rmarkdown")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        rmd   8.2Mb
    ```

# rmd

<details>

* Version: 0.1.4
* Source code: https://github.com/cran/rmd
* URL: https://github.com/pzhaonet/rmd
* BugReports: https://github.com/pzhaonet/rmd/issues
* Date/Publication: 2019-02-01 18:40:03 UTC
* Number of recursive dependencies: 122

Run `revdep_details(,"rmd")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘blogdown’ ‘bookdown’ ‘bookdownplus’ ‘citr’ ‘pagedown’ ‘rticles’
      ‘tinytex’ ‘xaringan’
      All declared Imports should be used.
    ```

# RnBeads

<details>

* Version: 2.0.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 219

Run `revdep_details(,"RnBeads")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
       
      1 Test Suite : 
      RnBeads RUnit Tests - 9 test functions, 1 error, 0 failures
      ERROR in /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R: Error while sourcing  /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R : Error in logger.error(txt) : 
        Missing required package missMethyl or its dependency
      
      Test files with failing tests
      
         test_differential.R 
           /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R 
      
      
      Error in BiocGenerics:::testPackage("RnBeads") : 
        unit tests failed for package RnBeads
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘IlluminaHumanMethylation450kmanifest’
    
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘cluster’ ‘ff’
      ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’ ‘matrixStats’
      ‘illuminaio’ ‘methylumi’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        R     3.1Mb
        doc   3.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘qvalue’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      Density Deviance Difference DoISVA Error EstDimRMT GenomeAxisTrack ID
      IdeogramTrack IlluminaHumanMethylation450kmanifest
      IlluminaHumanMethylationEPICmanifest Index Intensity Measure
      PairsBootRefFreeEwasModel Probe RGChannelSet Rd2HTML RefFreeEwasModel
      SNP Sample Slide Target Term UcscTrack Value addSex as.profileCGH
      assayDataElement assayDataElementNames barcode bv chrom combinedRank
      combinedRank.var comma covgMedian covgPercLow covgPercUp cv.glmnet
      daglad diffmeth diffmeth.p.adj.fdr diffmeth.p.val
      dinucleotideFrequency eps expectedCounts featureData featureData<-
      featureNames featureNames<- foreach geneCounts genome<- getCN
      getDoParWorkers getGreen getManifest getMeth getRed getSex getUnmeth
      getVarCov glmnet grid.draw grid.newpage group group1 group2 i
      impute.knn is.subsegmentation k letterFrequency lme loadRegionDB
      log10FDR log10P mapToGenome mclapply mean.diff mean.quot.log2 melt
      mergeRegionDBs muted n.sites ng.2 num.sites numSites numeric.names
      oddsRatios p.vals.t.na.adj pData percent_format phenoData phenoData<-
      plotTracks preprocessSWAN pvalues qvalue refText reg.type region.size
      registerDoParallel relative.coord report runLOLA samples seqlengths
      seqlevels<- sigCategories sites2ignore size solve.QP stopCluster sva
      target tsne type types universeCounts useMart v var.diff varFit
      varLabels x xmlValue y yint
    ```

# rosr

<details>

* Version: 0.0.6
* Source code: https://github.com/cran/rosr
* URL: https://github.com/pzhaonet/rosr
* BugReports: https://github.com/pzhaonet/rosr/issues
* Date/Publication: 2019-03-23 06:20:03 UTC
* Number of recursive dependencies: 108

Run `revdep_details(,"rosr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘bookdown’ ‘rstudioapi’ ‘tinytex’
      All declared Imports should be used.
    ```

# rpostgisLT

<details>

* Version: 0.6.0
* Source code: https://github.com/cran/rpostgisLT
* URL: https://github.com/mablab/rpostgisLT
* BugReports: https://github.com/mablab/rpostgisLT/issues
* Date/Publication: 2018-03-02 18:02:11 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"rpostgisLT")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# Rqc

<details>

* Version: 1.16.2
* Source code: ???
* URL: https://github.com/labbcb/Rqc
* BugReports: https://github.com/labbcb/Rqc/issues
* Date/Publication: 2018-11-03
* Number of recursive dependencies: 127

Run `revdep_details(,"Rqc")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# RQuantLib

<details>

* Version: 0.4.8
* Source code: https://github.com/cran/RQuantLib
* URL: http://dirk.eddelbuettel.com/code/rquantlib.html
* BugReports: https://github.com/eddelbuettel/rquantlib/issues
* Date/Publication: 2019-03-17 18:43:29 UTC
* Number of recursive dependencies: 62

Run `revdep_details(,"RQuantLib")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    +                fixFreq="Annual",
    +                floatFreq="Semiannual")
    + 
    + setEvaluationDate(as.Date("2016-2-16"))               
    + times<-times <- seq(0,14.75,.25)
    + dcurve <- DiscountCurve(params, tsQuotes, times=times,legparams)
    + 
    + # Price the Bermudan swaption
    + pricing <- AffineSwaption(params, dcurve,swaptionMaturities, swapTenors, volMatrix,legparams)
    + summary(pricing)
    + 
    + }    
    
     *** caught segfault ***
    address 0x0, cause 'memory not mapped'
    
    Traceback:
     1: affineWithRebuiltCurveEngine(params, matchlegs, c(ts$table$date),     ts$table$zeroRates, expiry, tenor, vol)
     2: AffineSwaption.default(params, dcurve, swaptionMaturities, swapTenors,     volMatrix, legparams)
     3: AffineSwaption(params, dcurve, swaptionMaturities, swapTenors,     volMatrix, legparams)
    An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/RQuantlib.R’ failed.
    Last 13 lines of output:
      +                             settleDate=as.Date('2002-2-15'),
      +                             dt=0.25,
      +                             interpWhat='discount', interpHow='loglinear')
      > discountCurve <- DiscountCurve(discountCurve.param, list(flat=0.05))
      > 
      > ZeroCouponBond(bond, discountCurve, dateparams)
      
       *** caught segfault ***
      address 0x0, cause 'memory not mapped'
      
      Traceback:
       1: ZeroBondWithRebuiltCurve(bond, c(discountCurve$table$date), discountCurve$table$zeroRates,     dateparams)
       2: ZeroCouponBond.default(bond, discountCurve, dateparams)
       3: ZeroCouponBond(bond, discountCurve, dateparams)
      An irrecoverable exception occurred. R is aborting now ...
    ```

# rsconnect

<details>

* Version: 0.8.13
* Source code: https://github.com/cran/rsconnect
* Date/Publication: 2019-01-10 19:50:03 UTC
* Number of recursive dependencies: 44

Run `revdep_details(,"rsconnect")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘keras’, ‘tensorflow’
    ```

# rtimicropem

<details>

* Version: 1.3
* Source code: https://github.com/cran/rtimicropem
* URL: http://github.com/ropensci/rtimicropem
* BugReports: http://github.com/ropensci/rtimicropem/issues
* Date/Publication: 2017-08-02 18:30:53 UTC
* Number of recursive dependencies: 85

Run `revdep_details(,"rtimicropem")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# RtutoR

<details>

* Version: 1.2
* Source code: https://github.com/cran/RtutoR
* Date/Publication: 2018-09-14 07:50:07 UTC
* Number of recursive dependencies: 108

Run `revdep_details(,"RtutoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# RxODE

<details>

* Version: 0.8.0-9
* Source code: https://github.com/cran/RxODE
* URL: https://www.r-project.org, https://github.com/nlmixrdevelopment/RxODE
* BugReports: https://github.com/nlmixrdevelopment/RxODE/issues
* Date/Publication: 2018-12-10 10:00:07 UTC
* Number of recursive dependencies: 108

Run `revdep_details(,"RxODE")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘n1qn1’
      All declared Imports should be used.
    ```

# sangerseqR

<details>

* Version: 1.18.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 38

Run `revdep_details(,"sangerseqR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    chromatogram,sangerseq: no visible global function definition for ‘IQR’
    chromatogram,sangerseq: no visible global function definition for ‘pdf’
    chromatogram,sangerseq: no visible global function definition for
      ‘plot’
    chromatogram,sangerseq: no visible global function definition for
      ‘rect’
    chromatogram,sangerseq: no visible global function definition for
      ‘lines’
    chromatogram,sangerseq: no visible global function definition for
      ‘mtext’
    chromatogram,sangerseq: no visible global function definition for
      ‘axis’
    chromatogram,sangerseq: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      IQR axis dev.off lines mtext par pdf plot quantile rect
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "axis", "lines", "mtext", "par", "plot", "rect")
      importFrom("stats", "IQR", "quantile")
    to your NAMESPACE file.
    ```

# santaR

<details>

* Version: 1.0
* Source code: https://github.com/cran/santaR
* URL: https://github.com/adwolfer
* Date/Publication: 2018-01-24 18:27:08 UTC
* Number of recursive dependencies: 67

Run `revdep_details(,"santaR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# SC3

<details>

* Version: 1.10.1
* Source code: ???
* URL: https://github.com/hemberg-lab/SC3
* BugReports: https://support.bioconductor.org/t/sc3/
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 115

Run `revdep_details(,"SC3")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    2.5Mb
    ```

# scone

<details>

* Version: 1.6.1
* Source code: ???
* BugReports: https://github.com/YosefLab/scone/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 188

Run `revdep_details(,"scone")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# SDEFSR

<details>

* Version: 0.7.1.0
* Source code: https://github.com/cran/SDEFSR
* URL: https://github.com/aklxao2/SDR
* Date/Publication: 2016-07-13 10:19:14
* Number of recursive dependencies: 49

Run `revdep_details(,"SDEFSR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# SEA

<details>

* Version: 1.0
* Source code: https://github.com/cran/SEA
* Date/Publication: 2018-05-24 12:35:11 UTC
* Number of recursive dependencies: 25

Run `revdep_details(,"SEA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        R   8.0Mb
    ```

# semdrw

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/semdrw
* Date/Publication: 2018-11-14 11:00:02 UTC
* Number of recursive dependencies: 121

Run `revdep_details(,"semdrw")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘lavaan’ ‘psych’ ‘semPlot’ ‘semTools’ ‘shinyAce’
      All declared Imports should be used.
    ```

# SEPA

<details>

* Version: 1.12.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 70

Run `revdep_details(,"SEPA")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘t.test’
    truetimepattern : <anonymous>: no visible global function definition
      for ‘p.adjust’
    truetimevisualize : <anonymous> : <anonymous>: no visible global
      function definition for ‘median’
    truetimevisualize: no visible binding for global variable ‘time’
    truetimevisualize: no visible binding for global variable ‘expmean’
    truetimevisualize: no visible binding for global variable ‘Gene’
    windowGOanalysis: no visible global function definition for ‘new’
    windowGOvisualize: no visible binding for global variable ‘Var1’
    windowGOvisualize: no visible binding for global variable ‘value’
    Undefined global functions or variables:
      Gene Var1 Var2 confint expmean fitted lines lm median new p.adjust
      pseudotime t.test time value xend yend
    Consider adding
      importFrom("graphics", "lines")
      importFrom("methods", "new")
      importFrom("stats", "confint", "fitted", "lm", "median", "p.adjust",
                 "t.test", "time")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# seqplots

<details>

* Version: 1.20.1
* Source code: ???
* URL: http://github.com/przemol/seqplots
* BugReports: http://github.com/przemol/seqplots/issues
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 109

Run `revdep_details(,"seqplots")` for more info

</details>

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'plotHeatmap'
      ‘plotz’ ‘FO’ ‘CL’ ‘sort_by’ ‘sort_mids’ ‘sort_dst’ ‘clst_mids’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.9Mb
      sub-directories of 1Mb or more:
        doc        3.7Mb
        seqplots  10.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiocManager’
      All declared Imports should be used.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("BWGFile_summary", ..., PACKAGE = "rtracklayer")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotHeatmap,list: no visible global function definition for ‘hclust’
    plotHeatmap,list: no visible global function definition for ‘dist’
    plotHeatmap,list: no visible global function definition for ‘cutree’
    plotHeatmap,list: no visible global function definition for
      ‘as.dendrogram’
    plotHeatmap,list: no visible global function definition for ‘title’
    Undefined global functions or variables:
      Var1 Var2 abline adjustcolor approx as.dendrogram axis box
      capture.output colorRampPalette cutree dist doFileOperations hclust
      image kmeans layout lines mtext par plot.new qt rainbow rect rgb text
      title value
    Consider adding
      importFrom("grDevices", "adjustcolor", "colorRampPalette", "rainbow",
                 "rgb")
      importFrom("graphics", "abline", "axis", "box", "image", "layout",
                 "lines", "mtext", "par", "plot.new", "rect", "text",
                 "title")
      importFrom("stats", "approx", "as.dendrogram", "cutree", "dist",
                 "hclust", "kmeans", "qt")
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# sglr

<details>

* Version: 0.7
* Source code: https://github.com/cran/sglr
* Date/Publication: 2014-05-03 00:54:29
* Number of recursive dependencies: 43

Run `revdep_details(,"sglr")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .computePStar: no visible global function definition for ‘uniroot’
    plotBoundary: no visible global function definition for
      ‘scale_y_continuous’
    Undefined global functions or variables:
      scale_y_continuous uniroot
    Consider adding
      importFrom("stats", "uniroot")
    to your NAMESPACE file.
    ```

# shiny.router

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/shiny.router
* Date/Publication: 2018-09-18 08:50:07 UTC
* Number of recursive dependencies: 21

Run `revdep_details(,"shiny.router")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# shiny.semantic

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/shiny.semantic
* BugReports: https://github.com/Appsilon/shiny.semantic/issues
* Date/Publication: 2018-05-11 10:38:06 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"shiny.semantic")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# shinyAce

<details>

* Version: 0.3.3
* Source code: https://github.com/cran/shinyAce
* BugReports: https://github.com/trestletech/shinyAce/issues
* Date/Publication: 2019-01-03 09:30:07 UTC
* Number of recursive dependencies: 31

Run `revdep_details(,"shinyAce")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        www   7.7Mb
    ```

# shinyaframe

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/shinyaframe
* Date/Publication: 2017-11-26 15:29:43 UTC
* Number of recursive dependencies: 46

Run `revdep_details(,"shinyaframe")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# shinyBS

<details>

* Version: 0.61
* Source code: https://github.com/cran/shinyBS
* URL: https://ebailey78.github.io/shinyBS
* BugReports: https://github.com/ebailey78/shinyBS/issues
* Date/Publication: 2015-03-31 07:52:38
* Number of recursive dependencies: 16

Run `revdep_details(,"shinyBS")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    popify: no visible global function definition for ‘runif’
    tipify: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

# shinyEffects

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/shinyEffects
* URL: https://github.com/DivadNojnarg/shinyEffects, https://divadnojnarg.github.io/shinyEffects/
* BugReports: https://github.com/DivadNojnarg/shinyEffects/issues
* Date/Publication: 2018-11-18 18:40:03 UTC
* Number of recursive dependencies: 28

Run `revdep_details(,"shinyEffects")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinydashboard’
      All declared Imports should be used.
    ```

# shinyHeatmaply

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/shinyHeatmaply
* URL: https://github.com/yonicd/shinyHeatmaply
* BugReports: https://github.com/yonicd/shinyHeatmaply/issues
* Date/Publication: 2017-03-28 06:11:58 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"shinyHeatmaply")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘htmlwidgets’ ‘jsonlite’ ‘viridis’
      All declared Imports should be used.
    ```

# shinyhelper

<details>

* Version: 0.3.1
* Source code: https://github.com/cran/shinyhelper
* BugReports: https://github.com/cwthom/shinyhelper/issues
* Date/Publication: 2018-10-28 16:20:02 UTC
* Number of recursive dependencies: 17

Run `revdep_details(,"shinyhelper")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# ShinyItemAnalysis

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/ShinyItemAnalysis
* BugReports: https://github.com/patriciamar/ShinyItemAnalysis/issues
* Date/Publication: 2019-02-25 16:40:02 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"ShinyItemAnalysis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘data.table’ ‘ggdendro’ ‘gridExtra’ ‘knitr’ ‘latticeExtra’ ‘msm’
      ‘plotly’ ‘shinyBS’ ‘shinydashboard’ ‘xtable’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘WrightMap’
    ```

# shinyjs

<details>

* Version: 1.0
* Source code: https://github.com/cran/shinyjs
* URL: https://deanattali.com/shinyjs
* BugReports: https://github.com/daattali/shinyjs/issues
* Date/Publication: 2018-01-08 22:21:13 UTC
* Number of recursive dependencies: 36

Run `revdep_details(,"shinyjs")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# shinyKGode

<details>

* Version: 1.0.5
* Source code: https://github.com/cran/shinyKGode
* URL: https://github.com/joewandy/shinyKGode
* Date/Publication: 2018-02-12 12:57:59 UTC
* Number of recursive dependencies: 50

Run `revdep_details(,"shinyKGode")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘KGode’ ‘XML’ ‘ggplot2’ ‘gridExtra’ ‘mvtnorm’ ‘reshape2’ ‘shinyjs’
      All declared Imports should be used.
    ```

# shinyMethyl

<details>

* Version: 1.18.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 131

Run `revdep_details(,"shinyMethyl")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘IlluminaHumanMethylation450kmanifest’
    
    Package suggested but not available for checking: ‘minfiData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# shinyrecap

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/shinyrecap
* URL: https://fellstat.github.io/shinyrecap/
* Date/Publication: 2019-01-19 23:40:03 UTC
* Number of recursive dependencies: 74

Run `revdep_details(,"shinyrecap")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘CARE1’ ‘LCMCR’ ‘coda’ ‘conting’ ‘dga’ ‘future’ ‘ggplot2’ ‘ipc’
      ‘promises’ ‘reshape’ ‘shinycssloaders’ ‘testthat’
      All declared Imports should be used.
    ```

# shinystan

<details>

* Version: 2.5.0
* Source code: https://github.com/cran/shinystan
* URL: http://mc-stan.org/, http://discourse.mc-stan.org
* BugReports: https://github.com/stan-dev/shinystan/issues/
* Date/Publication: 2018-05-01 03:31:02 UTC
* Number of recursive dependencies: 105

Run `revdep_details(,"shinystan")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘brms’
    ```

# shinyTANDEM

<details>

* Version: 1.20.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 25

Run `revdep_details(,"shinyTANDEM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘methods’ ‘mixtools’ ‘rTANDEM’ ‘shiny’ ‘xtable’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    stats.ui: no visible global function definition for ‘div’
    stats.ui: no visible global function definition for ‘h3’
    stats.ui: no visible global function definition for ‘uiOutput’
    tableAsHTML: no visible global function definition for ‘capture.output’
    tableAsHTML: no visible global function definition for ‘xtable’
    Undefined global functions or variables:
      GetResultsFromXML HTML abline actionButton addResourcePath at
      basicPage capture.output conditionalPanel dataset density div
      expect.value h2 h3 h4 h5 helpText htmlOutput includeHTML is isolate
      label legend like lines modified new normalmixEM num.peptides
      numericInput observe p pep.id plot plotOutput points prot.uid
      reactive reactiveValues renderPlot renderText renderUI runApp
      selectInput singleton span tabPanel tabsetPanel tag tagList tags
      textInput type uiOutput uid xtable
    Consider adding
      importFrom("graphics", "abline", "legend", "lines", "plot", "points")
      importFrom("methods", "is", "new")
      importFrom("stats", "density")
      importFrom("utils", "capture.output")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# sigmajs

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/sigmajs
* URL: http://sigmajs.john-coene.com/
* BugReports: https://github.com/JohnCoene/sigmajs/issues
* Date/Publication: 2019-04-09 11:10:09 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"sigmajs")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 28 marked UTF-8 strings
    ```

# SimDesign

<details>

* Version: 1.13
* Source code: https://github.com/cran/SimDesign
* URL: https://github.com/philchalmers/SimDesign, https://github.com/philchalmers/SimDesign/wiki
* Date/Publication: 2019-01-18 15:10:03 UTC
* Number of recursive dependencies: 71

Run `revdep_details(,"SimDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# singleCellTK

<details>

* Version: 1.2.3
* Source code: ???
* URL: https://compbiomed.github.io/sctk_docs/
* BugReports: https://github.com/compbiomed/singleCellTK/issues
* Date/Publication: 2018-12-01
* Number of recursive dependencies: 164

Run `revdep_details(,"singleCellTK")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘GSVAdata’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# soc.ca

<details>

* Version: 0.7.3
* Source code: https://github.com/cran/soc.ca
* URL: https://github.com/Rsoc/soc.ca
* Date/Publication: 2016-02-09 15:53:12
* Number of recursive dependencies: 46

Run `revdep_details(,"soc.ca")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 833 marked UTF-8 strings
    ```

# soilcarbon

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/soilcarbon
* URL: https://powellcenter-soilcarbon.github.io/soilcarbon/
* BugReports: https://github.com/powellcenter-soilcarbon/soilcarbon/issues
* Date/Publication: 2017-08-04 03:17:31 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"soilcarbon")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 145 marked UTF-8 strings
    ```

# soundgen

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/soundgen
* URL: http://cogsci.se/soundgen.html
* Date/Publication: 2019-03-13 06:30:03 UTC
* Number of recursive dependencies: 41

Run `revdep_details(,"soundgen")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘spectral’
    ```

# SpaDES.addins

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/SpaDES.addins
* URL: http://spades-addins.predictiveecology.org/, https://github.com/PredictiveEcology/SpaDES.addins
* BugReports: https://github.com/PredictiveEcology/SpaDES.addins/issues
* Date/Publication: 2019-02-03 20:13:20 UTC
* Number of recursive dependencies: 142

Run `revdep_details(,"SpaDES.addins")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘rstudioapi’
      All declared Imports should be used.
    ```

# SpatialEpiApp

<details>

* Version: 0.3
* Source code: https://github.com/cran/SpatialEpiApp
* URL: https://github.com/Paula-Moraga/SpatialEpiApp
* BugReports: https://github.com/Paula-Moraga/SpatialEpiApp/issues
* Date/Publication: 2017-08-28 22:10:33 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"SpatialEpiApp")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘SpatialEpi’ ‘dplyr’ ‘dygraphs’ ‘ggplot2’
      ‘htmlwidgets’ ‘knitr’ ‘leaflet’ ‘mapproj’ ‘maptools’ ‘rgdal’ ‘rgeos’
      ‘rmarkdown’ ‘shinyjs’ ‘spdep’ ‘xts’
      All declared Imports should be used.
    ```

# spectrolab

<details>

* Version: 0.0.8
* Source code: https://github.com/cran/spectrolab
* URL: https://github.com/meireles/spectrolab
* Date/Publication: 2018-10-28 15:00:03 UTC
* Number of recursive dependencies: 70

Run `revdep_details(,"spectrolab")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘usethis’
      All declared Imports should be used.
    ```

# SSDM

<details>

* Version: 0.2.4
* Source code: https://github.com/cran/SSDM
* URL: https://github.com/sylvainschmitt/SSDM
* BugReports: https://github.com/sylvainschmitt/SSDM/issues
* Date/Publication: 2018-01-29 18:50:19 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"SSDM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyFiles’
      All declared Imports should be used.
    ```

# sstModel

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/sstModel
* Date/Publication: 2018-05-03 22:21:08 UTC
* Number of recursive dependencies: 54

Run `revdep_details(,"sstModel")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinydashboard’
      All declared Imports should be used.
    ```

# STAT

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/STAT
* Date/Publication: 2019-04-01 16:40:10 UTC
* Number of recursive dependencies: 189

Run `revdep_details(,"STAT")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Hmisc’ ‘corrgram’ ‘datasets’ ‘dplyr’ ‘psych’ ‘psycho’ ‘rpivotTable’
      ‘shiny’
      All declared Imports should be used.
    ```

# steemr

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/steemr
* URL: https://github.com/pzhaonet/steemr
* BugReports: https://github.com/pzhaonet/steemr/issues
* Date/Publication: 2019-01-16 09:20:03 UTC
* Number of recursive dependencies: 94

Run `revdep_details(,"steemr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rlist’ ‘tm’ ‘zoo’
      All declared Imports should be used.
    ```

# stminsights

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/stminsights
* URL: https://github.com/cschwem2er/stminsights
* BugReports: https://github.com/cschwem2er/stminsights/issues
* Date/Publication: 2018-11-24 13:10:03 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"stminsights")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘huge’ ‘readr’ ‘scales’ ‘shinyjs’
      All declared Imports should be used.
    ```

# strataG

<details>

* Version: 2.0.2
* Source code: https://github.com/cran/strataG
* URL: https://github.com/EricArcher/strataG
* BugReports: https://github.com/EricArcher/strataG/issues
* Date/Publication: 2017-04-11 11:00:39 UTC
* Number of recursive dependencies: 127

Run `revdep_details(,"strataG")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘survival’
      All declared Imports should be used.
    ```

# subscreen

<details>

* Version: 2.0.1
* Source code: https://github.com/cran/subscreen
* Date/Publication: 2019-01-07 16:40:07 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"subscreen")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘V8’ ‘bsplus’ ‘colourpicker’ ‘dplyr’ ‘grDevices’ ‘graphics’
      ‘jsonlite’ ‘shinyjs’
      All declared Imports should be used.
    ```

# tableHTML

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/tableHTML
* URL: https://github.com/LyzandeR/tableHTML
* BugReports: https://github.com/LyzandeR/tableHTML/issues
* Date/Publication: 2019-03-16 17:30:02 UTC
* Number of recursive dependencies: 40

Run `revdep_details(,"tableHTML")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# tablerDash

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/tablerDash
* URL: https://rinterface.github.io/tablerDash/, https://github.com/RinteRface/tablerDash/
* BugReports: https://github.com/RinteRface/tablerDash/issues
* Date/Publication: 2019-03-08 16:00:03 UTC
* Number of recursive dependencies: 57

Run `revdep_details(,"tablerDash")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# TAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/TAShiny
* Date/Publication: 2018-11-27 17:30:03 UTC
* Number of recursive dependencies: 40

Run `revdep_details(,"TAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘SnowballC’ ‘dplyr’ ‘igraph’ ‘tm’ ‘wordcloud2’
      All declared Imports should be used.
    ```

# TCGAbiolinksGUI

<details>

* Version: 1.8.1
* Source code: ???
* Date/Publication: 2019-02-15
* Number of recursive dependencies: 288

Run `revdep_details(,"TCGAbiolinksGUI")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sesameData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tenXplore

<details>

* Version: 1.4.1
* Source code: ???
* Date/Publication: 2019-01-04
* Number of recursive dependencies: 92

Run `revdep_details(,"tenXplore")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    apd: warning in dir(system.file("app", package = "tenXplore"), full =
      TRUE): partial argument match of 'full' to 'full.names'
    ```

# testextra

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/testextra
* URL: https://github.com/RDocTaskForce/testextra
* BugReports: https://github.com/RDocTaskForce/testextra/issues
* Date/Publication: 2019-01-18 22:30:03 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"testextra")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘stringi’ ‘utils’
      All declared Imports should be used.
    ```

# TFutils

<details>

* Version: 1.2.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 193

Run `revdep_details(,"TFutils")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘TFutils-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: genemodelDF
    > ### Title: use EnsDb to generate an exon-level model of genes identified by
    > ###   symbol
    > ### Aliases: genemodelDF
    > 
    > ### ** Examples
    > 
    > if (requireNamespace("EnsDb.Hsapiens.v75")) {
    +  orm = genemodelDF("ORMDL3", EnsDb.Hsapiens.v75::EnsDb.Hsapiens.v75)
    +  dim(orm)
    + }
    Loading required namespace: EnsDb.Hsapiens.v75
    Failed with error:  'there is no package called 'EnsDb.Hsapiens.v75''
    > head(orm)
    Error in head(orm) : object 'orm' not found
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘EnsDb.Hsapiens.v75’
    ```

*   checking R code for possible problems ... NOTE
    ```
    directHitsInCISBP: no visible binding for global variable
      ‘DISEASE.TRAIT’
    directHitsInCISBP: no visible binding for global variable ‘MAPPED_GENE’
    directHitsInCISBP: no visible binding for global variable ‘cisbpTFcat’
    directHitsInCISBP: no visible binding for global variable ‘HGNC’
    directHitsInCISBP: no visible binding for global variable ‘Family_Name’
    topTraitsOfTargets: no visible global function definition for
      ‘geneIdType’
    topTraitsOfTargets: no visible global function definition for
      ‘mapIdentifiers’
    topTraitsOfTargets: no visible global function definition for
      ‘SymbolIdentifier’
    topTraitsOfTargets: no visible global function definition for ‘geneIds’
    topTraitsOfTargets: no visible global function definition for ‘mcols’
    topTraitsOfTargets: no visible binding for global variable
      ‘DISEASE.TRAIT’
    Undefined global functions or variables:
      DISEASE.TRAIT Family_Name HGNC MAPPED_GENE SymbolIdentifier
      cisbpTFcat geneIdType geneIds mapIdentifiers mcols
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# timeline

<details>

* Version: 0.9
* Source code: https://github.com/cran/timeline
* URL: http://jason.bryer.org/timeline, http://github.com/jbryer/timeline
* BugReports: https://github.com/jbryer/timeline/issues
* Date/Publication: 2013-05-09 17:03:14
* Number of recursive dependencies: 43

Run `revdep_details(,"timeline")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘shiny’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    timeline: no visible global function definition for ‘ggplot’
    timeline: no visible global function definition for ‘geom_segment’
    timeline: no visible global function definition for ‘aes_string’
    timeline: no visible global function definition for ‘geom_rect’
    timeline: no visible global function definition for ‘geom_text’
    timeline: no visible global function definition for ‘theme’
    timeline: no visible global function definition for ‘element_blank’
    timeline: no visible global function definition for ‘xlab’
    timeline: no visible global function definition for ‘ylab’
    timeline: no visible global function definition for ‘xlim’
    timeline: no visible global function definition for
      ‘scale_y_continuous’
    timeline: no visible global function definition for ‘geom_point’
    timeline: no visible global function definition for ‘scale_color_grey’
    timeline: no visible global function definition for ‘geom_hline’
    Undefined global functions or variables:
      aes_string element_blank geom_hline geom_point geom_rect geom_segment
      geom_text ggplot scale_color_grey scale_y_continuous theme xlab xlim
      ylab
    ```

# TimerQuant

<details>

* Version: 1.12.0
* Source code: ???
* Date/Publication: 2018-11-01
* Number of recursive dependencies: 63

Run `revdep_details(,"TimerQuant")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rainbow’
    plotPrimordiumProfile: no visible binding for global variable ‘median’
    plotPrimordiumProfile: no visible binding for global variable ‘mad’
    plotPrimordiumProfile: no visible global function definition for ‘par’
    plotPrimordiumProfile: no visible global function definition for ‘plot’
    plotPrimordiumProfile: no visible global function definition for ‘axis’
    plotPrimordiumProfile: no visible global function definition for
      ‘points’
    plotPrimordiumProfile: no visible global function definition for
      ‘polygon’
    plotPrimordiumProfile: no visible global function definition for ‘rgb’
    simulatedRatio: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      approxfun axis mad median optimize par plot points polygon predict
      rainbow rgb rnorm
    Consider adding
      importFrom("grDevices", "rainbow", "rgb")
      importFrom("graphics", "axis", "par", "plot", "points", "polygon")
      importFrom("stats", "approxfun", "mad", "median", "optimize",
                 "predict", "rnorm")
    to your NAMESPACE file.
    ```

# timeseriesdb

<details>

* Version: 0.4.1
* Source code: https://github.com/cran/timeseriesdb
* URL: https://github.com/mbannert/timeseriesdb
* BugReports: https://github.com/mbannert/timeseriesdb/issues
* Date/Publication: 2018-08-06 11:20:03 UTC
* Number of recursive dependencies: 38

Run `revdep_details(,"timeseriesdb")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# timevis

<details>

* Version: 0.5
* Source code: https://github.com/cran/timevis
* URL: https://github.com/daattali/timevis, http://daattali.com/shiny/timevis-demo/
* BugReports: https://github.com/daattali/timevis/issues
* Date/Publication: 2019-01-16 09:00:03 UTC
* Number of recursive dependencies: 40

Run `revdep_details(,"timevis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

# tmaptools

<details>

* Version: 2.0-1
* Source code: https://github.com/cran/tmaptools
* URL: https://github.com/mtennekes/tmaptools
* BugReports: https://github.com/mtennekes/tmaptools/issues
* Date/Publication: 2018-08-07 08:20:03 UTC
* Number of recursive dependencies: 107

Run `revdep_details(,"tmaptools")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘osmar’
    ```

# trackeRapp

<details>

* Version: 1.0
* Source code: https://github.com/cran/trackeRapp
* URL: https://github.com/trackerproject/trackeRapp
* BugReports: https://github.com/trackerproject/trackeRapp/issues
* Date/Publication: 2019-04-12 08:02:41 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"trackeRapp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘V8’ ‘colorspace’
      All declared Imports should be used.
    ```

# tricolore

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/tricolore
* Date/Publication: 2018-09-13 22:50:02 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"tricolore")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 88 marked UTF-8 strings
    ```

# TSCAN

<details>

* Version: 1.20.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 59

Run `revdep_details(,"TSCAN")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘lm’
    exprmclust: no visible global function definition for ‘dist’
    plotmclust: no visible binding for global variable ‘pca_dim_1’
    plotmclust: no visible binding for global variable ‘pca_dim_2’
    plotmclust: no visible binding for global variable ‘sample_name’
    preprocess: no visible binding for global variable ‘sd’
    preprocess: no visible global function definition for ‘hclust’
    preprocess: no visible global function definition for ‘dist’
    preprocess: no visible global function definition for ‘cutree’
    preprocess: no visible global function definition for ‘aggregate’
    singlegeneplot: no visible global function definition for
      ‘fitted.values’
    singlegeneplot: no visible binding for global variable ‘predict’
    Undefined global functions or variables:
      aggregate cutree dist fitted.values hclust lm p.adjust pca_dim_1
      pca_dim_2 pchisq prcomp predict sample_name sd
    Consider adding
      importFrom("stats", "aggregate", "cutree", "dist", "fitted.values",
                 "hclust", "lm", "p.adjust", "pchisq", "prcomp", "predict",
                 "sd")
    to your NAMESPACE file.
    ```

# ttestshiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/ttestshiny
* Date/Publication: 2018-10-30 16:30:03 UTC
* Number of recursive dependencies: 30

Run `revdep_details(,"ttestshiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘shinyAce’ ‘shinyjs’
      All declared Imports should be used.
    ```

# TVTB

<details>

* Version: 1.8.0
* Source code: ???
* URL: https://github.com/kevinrue/TVTB
* BugReports: https://github.com/kevinrue/TVTB/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 135

Run `revdep_details(,"TVTB")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
        rs1426654 rs150379789 rs570906312 rs538198029 rs553496066 rs574775672 
             TRUE        TRUE       FALSE       FALSE        TRUE       FALSE 
      rs140666229 rs556950130 rs575303689 rs147513140 rs187525777 rs192454382 
             TRUE       FALSE        TRUE        TRUE       FALSE       FALSE 
      rs184818838 rs566886499 rs199924625 rs555872528 rs531820822 rs201353600 
            FALSE       FALSE       FALSE       FALSE       FALSE       FALSE 
      rs550201688 rs565338261 rs201239799 rs200461129 rs146726548 
            FALSE       FALSE       FALSE       FALSE        TRUE 
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 225 SKIPPED: 2 FAILED: 2
      1. Error: all signatures work to completion (@test_plotInfo-methods.R#25) 
      2. Error: invalid metric/phenotype combination is detected (@test_plotInfo-methods.R#50) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘EnsDb.Hsapiens.v75’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R     2.1Mb
        doc   2.2Mb
    ```

# twoddpcr

<details>

* Version: 1.6.0
* Source code: ???
* URL: http://github.com/CRUKMI-ComputationalBiology/twoddpcr/
* BugReports: http://github.com/CRUKMI-ComputationalBiology/twoddpcr/issues/
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 91

Run `revdep_details(,"twoddpcr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    4.1Mb
    ```

# Ularcirc

<details>

* Version: 1.0.0
* Source code: ???
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 111

Run `revdep_details(,"Ularcirc")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mirbase.db’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# VariantFiltering

<details>

* Version: 1.18.0
* Source code: ???
* URL: https://github.com/rcastelo/VariantFiltering
* BugReports: https://github.com/rcastelo/VariantFiltering/issues
* Date/Publication: 2018-10-30
* Number of recursive dependencies: 142

Run `revdep_details(,"VariantFiltering")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: VariantFilteringParam-class
    > ### Title: VariantFiltering parameter class
    > ### Aliases: sequence_variant.gSOXP class:VariantFilteringParam
    > ###   VariantFilteringParam VariantFilteringParam-class
    > ###   show,VariantFilteringParam-method $,VariantFilteringParam-method
    > ###   names,VariantFilteringParam-method
    > ###   filters,VariantFilteringParam-method
    > ###   cutoffs,VariantFilteringParam-method sog,VariantFilteringParam-method
    > ###   spliceSiteMatricesHuman variantLocations
    > ### Keywords: classes,methods
    > 
    > ### ** Examples
    > 
    > vfpar <- VariantFilteringParam(system.file("extdata", "CEUtrio.vcf.bgz", package="VariantFiltering"),
    +                                system.file("extdata", "CEUtrio.ped", package="VariantFiltering"),
    +                                snpdb=character(0), otherAnnotations=character(0))
    Error in .loadAnnotationPackageObject(bsgenome, "bsgenome", "BSgenome") : 
      please install the Bioconductor package BSgenome.Hsapiens.1000genomes.hs37d5.
    Calls: VariantFilteringParam -> .loadAnnotationPackageObject
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      Test files with failing tests
      
         test_VariantFilteringParam-methods.R 
           test_VariantFilteringParam 
      
         test_inheritance-models.R 
           /Users/barret/odrive/AmazonCloudDrive/git/rstudio/shiny/shiny/revdep/checks.noindex/VariantFiltering/new/VariantFiltering.Rcheck/VariantFiltering/unitTests/test_inheritance-models.R 
      
         test_location-methods.R 
           test_location_annotations 
      
      
      Error in BiocGenerics:::testPackage("VariantFiltering") : 
        unit tests failed for package VariantFiltering
      Execution halted
    ```

*   checking running R code from vignettes ...
    ```
       ‘usingVariantFiltering.Rnw’ ... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘usingVariantFiltering.Rnw’
      ...
    
    
    > CEUvcf <- file.path(system.file("extdata", package = "VariantFiltering"), 
    +     "CEUtrio.vcf.bgz")
    
    > vfpar <- VariantFilteringParam(CEUvcf)
    
      When sourcing 'usingVariantFiltering.R':
    Error: please install the Bioconductor package BSgenome.Hsapiens.1000genomes.hs37d5.
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘BSgenome.Hsapiens.1000genomes.hs37d5’
      ‘SNPlocs.Hsapiens.dbSNP144.GRCh37’ ‘MafDb.1Kgenomes.phase1.hs37d5’
      ‘phastCons100way.UCSC.hg19’ ‘PolyPhen.Hsapiens.dbSNP131’
      ‘SIFT.Hsapiens.dbSNP137’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        extdata   3.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'S4Vectors:::labeledLine' 'VariantAnnotation:::.checkArgs'
      'VariantAnnotation:::.consolidateHits'
      'VariantAnnotation:::.returnEmpty'
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      '.adjustForStrandSense' '.ancestorsSO' '.findSOIDs'
    ```

*   checking R code for possible problems ... NOTE
    ```
    annotateVariants,MafDb: no visible global function definition for
      'mafByOverlaps'
    Undefined global functions or variables:
      mafByOverlaps
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘phastCons100way.UCSC.hg19’
    ```

# vdiffr

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/vdiffr
* URL: https://github.com/r-lib/vdiffr
* BugReports: https://github.com/r-lib/vdiffr/issues
* Date/Publication: 2019-01-02 19:30:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(,"vdiffr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘freetypeharfbuzz’
      All declared Imports should be used.
    ```

# visNetwork

<details>

* Version: 2.0.6
* Source code: https://github.com/cran/visNetwork
* URL: http://datastorm-open.github.io/visNetwork/
* BugReports: https://github.com/datastorm-open/visNetwork/issues
* Date/Publication: 2019-03-26 15:10:03 UTC
* Number of recursive dependencies: 78

Run `revdep_details(,"visNetwork")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc           4.2Mb
        docjs         1.4Mb
        htmlwidgets   3.9Mb
    ```

# wallace

<details>

* Version: 1.0.6
* Source code: https://github.com/cran/wallace
* URL: https://wallaceEcoMod.github.io
* Date/Publication: 2018-10-07 20:10:03 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"wallace")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ENMeval’ ‘RColorBrewer’ ‘XML’ ‘dismo’ ‘dplyr’ ‘leaflet.extras’
      ‘maptools’ ‘raster’ ‘rgdal’ ‘rgeos’ ‘rmarkdown’ ‘shinyjs’
      ‘shinythemes’ ‘spThin’ ‘spocc’ ‘zip’
      All declared Imports should be used.
    ```

# wiseR

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/wiseR
* URL: https://github.com/SAFE-ICU/wiseR
* BugReports: https://github.com/SAFE-ICU/wiseR/issues
* Date/Publication: 2018-11-29 16:10:03 UTC
* Number of recursive dependencies: 127

Run `revdep_details(,"wiseR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        bn   7.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘DescTools’ ‘HydeNet’ ‘RBGL’ ‘Rgraphviz’ ‘arules’ ‘bnlearn’
      ‘dplyr’ ‘graph’ ‘igraph’ ‘linkcomm’ ‘missRanger’ ‘parallel’ ‘psych’
      ‘rhandsontable’ ‘rintrojs’ ‘shinyBS’ ‘shinyWidgets’ ‘shinyalert’
      ‘shinycssloaders’ ‘shinydashboard’ ‘tools’ ‘visNetwork’
      All declared Imports should be used.
    ```

# wTO

<details>

* Version: 1.6.3
* Source code: https://github.com/cran/wTO
* Date/Publication: 2018-11-17 11:50:03 UTC
* Number of recursive dependencies: 38

Run `revdep_details(,"wTO")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# yuimaGUI

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/yuimaGUI
* Date/Publication: 2018-11-25 12:50:03 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"yuimaGUI")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ghyp’
      All declared Imports should be used.
    ```

# zscorer

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/zscorer
* URL: https://github.com/nutriverse/zscorer
* BugReports: https://github.com/nutriverse/zscorer/issues
* Date/Publication: 2018-11-26 13:10:02 UTC
* Number of recursive dependencies: 52

Run `revdep_details(,"zscorer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

